<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.39">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Zhehao Hu">
<meta name="author" content="Jenny Neuhaus">
<meta name="keywords" content="Integrative taxonomy, Isopoda, Deep-sea, Biogeography">

<title>Integrative Taxonomy of North Atlantic Acanthocope with description of a new species</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-57a92833092fd6dc0d7710b6bdfff9c5.css" rel="stylesheet" class="quarto-color-scheme" id="quarto-text-highlighting-styles">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-dark-57a92833092fd6dc0d7710b6bdfff9c5.css" rel="prefetch" class="quarto-color-scheme quarto-color-alternate" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-c10486ce9d3a69006888ae96cd0f031c.min.css" rel="stylesheet" append-hash="true" class="quarto-color-scheme" id="quarto-bootstrap" data-mode="light">
<link href="site_libs/bootstrap/bootstrap-dark-b77dd1961d0a715ecd19962e84856c94.min.css" rel="prefetch" append-hash="true" class="quarto-color-scheme quarto-color-alternate" id="quarto-bootstrap" data-mode="dark">
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>


<meta name="citation_title" content="Integrative Taxonomy of North Atlantic *Acanthocope* with description of a new species">
<meta name="citation_keywords" content="Integrative taxonomy,Isopoda,Deep-sea,Biogeography">
<meta name="citation_author" content="Zhehao Hu">
<meta name="citation_author" content="Jenny Neuhaus">
<meta name="citation_language" content="en">
<meta name="citation_reference" content="citation_title=Application of Microsatellite Markers in Conservation Genetics and Fisheries Management: Recent Advances in Population Structure Analysis and Conservation Strategies;,citation_abstract=Microsatellites are the most popular and versatile genetic marker with myriads of applications in population genetics, conservation biology, and evolutionary biology. These are the arrays of DNA sequences, consisting of tandemly repeating mono-, di-, tri-, and tetranucleotide units, which are distributed throughout the genomes of most eukaryotic species. Microsatellites are codominant in nature, highly polymorphic, easily typed, and Mendelian inherited, all properties which make them very suitable for the study of population structure and pedigree analysis and capable of detecting differences among closely related species. PCR for microsatellites can be automated for identifying simple sequence repeat polymorphism. Small amount of blood samples or alcohol preserved tissue is adequate for analyzing them. Most of the microsatellites are noncoding, and therefore variations are independent of natural selection. These properties make microsatellites ideal genetic markers for conservation genetics and fisheries management. This review addresses the applications of microsatellite markers in conservation genetics and recent advances in population structure analysis in the context of fisheries management.;,citation_author=P. M. Abdul-Muneer;,citation_publication_date=2014-04;,citation_cover_date=2014-04;,citation_year=2014;,citation_fulltext_html_url=https://www.hindawi.com/journals/gri/2014/691759/;,citation_doi=10.1155/2014/691759;,citation_issn=2090-3154, 2090-3162;,citation_volume=2014;,citation_language=en-US;,citation_journal_title=Genetics Research International;">
<meta name="citation_reference" content="citation_title=Phylogeny in cryptic weevils: Molecules, morphology and new genera of western Palaearctic Cryptorhynchinae (Coleoptera:Curculionidae);,citation_abstract=A phylogeny is presented for the western Palaearctic representatives of the weevil subfamily Cryptorhynchinae using a combination of phenotypic and genotypic characters. This phylogeny is the first for the extremely species-rich Cryptorhynchinae to use molecular data (mitochondrial CO1 and 16S as well as nuclear ribosomal 28S). The results of this study show the need for molecular tools within this morphologically cryptic group of weevils and provide a scaffold based on which genus assignment can be tested. The present study mostly corroborates the current subdivision into genera (but many of the subgeneric groups are questioned). Three new genera are described: Montanacalles gen. nov. (type species: Kyklioacalles nevadaensis Stüben, 2001), Coloracalles gen. nov. (type species: Acalles humerosus Fairmaire, 1862) and Elliptacalles gen. nov. (type species: Acalles longus Desbrochers, 1892). Relevant external characters and the male genitalia of all discussed taxa are illustrated. Three species are transferred to different genera: Kyklioacalles aubei (Boheman, 1837) (formerly: Acalles), Ruteria major (Solari A. &amp;amp;amp; F., 1907) and Ruteria minosi (Bahr &amp; Bayer, 2005) (both formerly Echinodera).;,citation_author=Jonas J. Astrin;,citation_author=Peter E. Stüben;,citation_publication_date=2008;,citation_cover_date=2008;,citation_year=2008;,citation_fulltext_html_url=http://www.publish.csiro.au/?paper=IS07057;,citation_issue=5;,citation_doi=10.1071/IS07057;,citation_issn=1445-5226;,citation_volume=22;,citation_language=en-US;,citation_journal_title=Invert. Systematics;">
<meta name="citation_reference" content="citation_title=Applied Multivariate Statistics in R;,citation_author=Jonathan D Bakker;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_fulltext_html_url=https://uw.pressbooks.pub/appliedmultivariatestatistics/;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Preliminary Notice of the Isopoda collected during the Voyage of H.M.S. &amp;amp;amp;quot;Challenger&amp;quot;. Part 2. Munnopsidae;,citation_author=F E Beddard;,citation_publication_date=1885;,citation_cover_date=1885;,citation_year=1885;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Report on the Isopoda collected by H.M.S. Challenger during the Years 1873-1876. Second Part;,citation_author=Frank Evers Beddard;,citation_publication_date=1886;,citation_cover_date=1886;,citation_year=1886;,citation_fulltext_html_url=http://19thcenturyscience.org/HMSC/HMSC-Reports/Zool-48/README.htm;,citation_volume=17;,citation_journal_title=H.M.S. Challenger Reports;">
<meta name="citation_reference" content="citation_title=Cryptic species as a window on diversity and conservation;,citation_author=David Bickford;,citation_author=David J. Lohman;,citation_author=Navjot S. Sodhi;,citation_author=Peter K. L. Ng;,citation_author=Rudolf Meier;,citation_author=Kevin Winker;,citation_author=Krista K. Ingram;,citation_author=Indraneil Das;,citation_publication_date=2007-03;,citation_cover_date=2007-03;,citation_year=2007;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0169534706003703;,citation_issue=3;,citation_doi=10.1016/j.tree.2006.11.004;,citation_issn=01695347;,citation_volume=22;,citation_language=en-US;,citation_journal_title=Trends in Ecology &amp;amp;amp; Evolution;">
<meta name="citation_reference" content="citation_title=Does the Mid-Atlantic Ridge affect the distribution of abyssal benthic crustaceans across the Atlantic Ocean?;,citation_abstract=A trans-Atlantic transect along the Vema Fracture Zone was sampled during the Vema-TRANSIT expedition in 2014/15. The aim of the cruise was to investigate whether the Mid-Atlantic Ridge (MAR) isolates the abyssal fauna of the western and eastern abyssal basins.;,citation_author=Simon Bober;,citation_author=Saskia Brix;,citation_author=Torben Riehl;,citation_author=Martin Schwentner;,citation_author=Angelika Brandt;,citation_publication_date=2018-02;,citation_cover_date=2018-02;,citation_year=2018;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S096706451630412X;,citation_doi=10.1016/j.dsr2.2018.02.007;,citation_issn=09670645;,citation_volume=148;,citation_language=en-US;,citation_journal_title=Deep Sea Research Part II: Topical Studies in Oceanography;">
<meta name="citation_reference" content="citation_title=Le genre Acanthocope Beddard (Isopode Asellote) : description de deux espèces nouvelles. Remarques taxinomiques et biogéographiques;,citation_author=Pierre Chardy;,citation_publication_date=1972;,citation_cover_date=1972;,citation_year=1972;,citation_fulltext_html_url=https://www.biodiversitylibrary.org/part/272572;,citation_issue=30;,citation_doi=10.5962/p.272572;,citation_issn=0300-9386;,citation_volume=36;,citation_language=fr-FR;,citation_journal_title=Bulletin du Muséum national d’histoire naturelle;">
<meta name="citation_reference" content="citation_title=Non-parametric multivariate analyses of changes in community structure;,citation_abstract=Abstract In the early 1980s, a strategy for graphical representation of multivariate (multi-species) abundance data was introduced into marine ecology by, among others, Field, et al. (1982). A decade on, it is instructive to: (i) identify which elements of this often-quoted strategy have proved most useful in practical assessment of community change resulting from pollution impact; and (ii) ask to what extent evolution of techniques in the intervening years has added self-consistency and comprehensiveness to the approach. The pivotal concept has proved to be that of a biologically-relevant definition of similarity of two samples, and its utilization mainly in simple rank form, for example “sample A is more similar to sample B than it is to sample C”. Statistical assumptions about the data are thus minimized and the resulting non-parametric techniques will be of very general applicability. From such a starting point, a unified framework needs to encompass: (i) the display of community patterns through clustering and ordination of samples; (ii) identification of species principally responsible for determining sample groupings; (iii) statistical tests for differences in space and time (multivariate analogues of analysis of variance, based on rank similarities); and (iv) the linking of community differences to patterns in the physical and chemical environment (the latter also dictated by rank similarities between samples). Techniques are described that bring such a framework into place, and areas in which problems remain are identified. Accumulated practical experience with these methods is discussed, in particular applications to marine benthos, and it is concluded that they have much to offer practitioners of environmental impact studies on communities.;,citation_author=K. R. Clarke;,citation_publication_date=1993-03;,citation_cover_date=1993-03;,citation_year=1993;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/j.1442-9993.1993.tb00438.x;,citation_issue=1;,citation_doi=10.1111/j.1442-9993.1993.tb00438.x;,citation_issn=0307-692X;,citation_volume=18;,citation_language=en-US;,citation_journal_title=Australian Journal of Ecology;">
<meta name="citation_reference" content="citation_title=Disentangling microbial networks across pelagic zones in the tropical and subtropical global ocean;,citation_abstract=Abstract Microbial interactions are vital in maintaining ocean ecosystem function, yet their dynamic nature and complexity remain largely unexplored. Here, we use association networks to investigate possible ecological interactions in the marine microbiome among archaea, bacteria, and picoeukaryotes throughout different depths and geographical regions of the tropical and subtropical global ocean. Our findings reveal that potential microbial interactions change with depth and geographical scale, exhibiting highly heterogeneous distributions. A few potential interactions were global, meaning they occurred across regions at the same depth, while 11-36% were regional within specific depths. The bathypelagic zone had the lowest proportion of global associations, and regional associations increased with depth. Moreover, we observed that most surface water associations do not persist in deeper ocean layers despite microbial vertical dispersal. Our work contributes to a deeper understanding of the tropical and subtropical global ocean interactome, which is essential for addressing the challenges posed by global change.;,citation_author=Ina M. Deutschmann;,citation_author=Erwan Delage;,citation_author=Caterina R. Giner;,citation_author=Marta Sebastián;,citation_author=Julie Poulain;,citation_author=Javier Arístegui;,citation_author=Carlos M. Duarte;,citation_author=Silvia G. Acinas;,citation_author=Ramon Massana;,citation_author=Josep M. Gasol;,citation_author=Damien Eveillard;,citation_author=Samuel Chaffron;,citation_author=Ramiro Logares;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-023-44550-y;,citation_issue=1;,citation_doi=10.1038/s41467-023-44550-y;,citation_issn=2041-1723;,citation_volume=15;,citation_language=en-US;,citation_journal_title=Nat Commun;">
<meta name="citation_reference" content="citation_title=Identifying Causes of Patterns in Ecological Networks: Opportunities and Limitations;,citation_abstract=Ecological networks depict the interactions between species, mainly based on observations in the field. The information contained in such interaction matrices depends on the sampling design, and typically, compounds preferences (specialization) and abundances (activity). Null models are the primary vehicles to disentangle the effects of specialization from those of sampling and abundance, but they ignore the feedback of network structure on abundances. Hence, network structure, as exemplified here by modularity, is difficult to link to specific causes. Indeed, various processes lead to modularity and to specific interaction patterns more generally. Inferring (co)evolutionary dynamics is even more challenging, as competition and trait matching yield identical patterns of interactions. A satisfactory resolution of the underlying factors determining network structure will require substantial additional information, not only on independently assessed abundances, but also on traits, and ideally on fitness consequences as measured in experimental setups.;,citation_author=Carsten F. Dormann;,citation_author=Jochen Fründ;,citation_author=H. Martin Schaefer;,citation_publication_date=2017-11;,citation_cover_date=2017-11;,citation_year=2017;,citation_fulltext_html_url=https://www.annualreviews.org/doi/10.1146/annurev-ecolsys-110316-022928;,citation_issue=1;,citation_doi=10.1146/annurev-ecolsys-110316-022928;,citation_issn=1543-592X, 1545-2069;,citation_volume=48;,citation_language=en-US;,citation_journal_title=Annu. Rev. Ecol. Evol. Syst.;">
<meta name="citation_reference" content="citation_title=Parasites of crustaceans (Isopoda: Bopyridae) evolved from fish parasites: Molecular and morphological evidence;,citation_author=Hermann Dreyer;,citation_author=Wägele Johann-Wolfgang;,citation_publication_date=2001-01;,citation_cover_date=2001-01;,citation_year=2001;,citation_issue=3;,citation_volume=103;,citation_journal_title=Zoology;">
<meta name="citation_reference" content="citation_title=Bayesian phylogeographic inferences reveal contrasting colonization dynamics among &amp;amp;amp;lt;span style=&amp;quot;font-variant:small-caps;&amp;quot;&amp;gt;E&amp;lt;/span&amp;gt; uropean groundwater isopods;,citation_abstract=Abstract The potentially important role of northern microrefugia during postglacial dispersal is challenging the view of southern E urope as a refuge and source area of E uropean biota. In groundwaters, large geographic ranges of presumably good dispersers are increasingly suspected to consist of assemblages of cryptic species with narrow ranges. Moreover, a large species range, even when confirmed by molecular evidence, tells us little about the spatiotemporal dynamics of dispersal. Here, we used phylogenetic inferences, species delineation methods and B ayesian phylogeographic diffusion models to test for the likelihood of postglacial colonization from distant refugia among five morphospecies of P roasellus ( I sopoda, A sellidae). All morphospecies except one were monophyletic, but they comprised a total of 15–17 cryptic species. Three cryptic species retained ranges that spanned a distance $&amp;amp;gt;$650&nbsp;km, similar to that of the nominal morphospecies. Bayesian diffusion models based on mitochondrial markers revealed considerable spatiotemporal heterogeneity in dispersal rates, suggesting that short-time dispersal windows were instrumental in shaping species ranges. Only one species was found to experience a recent, presumably postglacial, range expansion. The J ura and A lpine foothills probably played a major role in maintaining diversity within P roasellus in northern regions by acting both as diversification hotspots and P leistocene refugia. Gaining insight into the spatiotemporal heterogeneity of dispersal rates revealed contrasting colonization dynamics among species that were not consistent with a global postglacial colonization of E urope from distant refugia.;,citation_author=D. Eme;,citation_author=F. Malard;,citation_author=L. Konecny-Dupré;,citation_author=T. Lefébure;,citation_author=C. J. Douady;,citation_publication_date=2013-11;,citation_cover_date=2013-11;,citation_year=2013;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/mec.12520;,citation_issue=22;,citation_doi=10.1111/mec.12520;,citation_issn=0962-1083, 1365-294X;,citation_volume=22;,citation_language=en-US;,citation_journal_title=Molecular Ecology;">
<meta name="citation_reference" content="citation_title=Challenges and Advances in the Taxonomy of Deep-Sea Peracarida: From Traditional to Modern Methods;,citation_abstract=As one of the oldest branches of biology, taxonomy deals with the identification, classification and naming of living organisms, using a variety of tools to explore traits at the morphological and molecular level. In the deep sea, particular challenges are posed to the taxonomic differentiation of species. Relatively limited sampling effort coupled with apparent high diversity, compared to many other marine environments, means that many species sampled are undescribed, and few specimens are available for each putative species. The resulting scarce knowledge of intraspecific variation makes it difficult to recognize species boundaries and thus to assess the actual diversity and distribution of species. In this review article, we highlight some of these challenges in deep-sea taxonomy using the example of peracarid crustaceans. Specifically, we offer a detailed overview of traditional as well as modern methods that are used in the taxonomic analysis of deep-sea Peracarida. Furthermore, methods are presented that have not yet been used in peracarid taxonomy, but have potential for the analysis of internal and external structures in the future. The focus of this compilation is on morphological methods for the identification, delimitation and description of species, with references to molecular analysis included where relevant, as these methods are an indispensable part of an integrative taxonomic approach. The taxonomic impediment, i.e. the shortage of taxonomists in view of a high undescribed biodiversity, is discussed in the context of the existing large taxonomic knowledge gaps in connection with the increasing threat to deep-sea ecosystems. Whilst peracarid crustaceans are used here as an exemplary taxon, the methodology described has broad relevance to many other deep-sea taxa, and thus will support broader research into deep-sea biodiversity and ecology more widely.;,citation_author=Inmaculada Frutos;,citation_author=Stefanie Kaiser;,citation_author=Łukasz Pułaski;,citation_author=Maciej Studzian;,citation_author=Magdalena Błażewicz;,citation_publication_date=2022-06;,citation_cover_date=2022-06;,citation_year=2022;,citation_fulltext_html_url=https://www.frontiersin.org/articles/10.3389/fmars.2022.799191/full;,citation_doi=10.3389/fmars.2022.799191;,citation_issn=2296-7745;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Front. Mar. Sci.;">
<meta name="citation_reference" content="citation_title=Redesign of &amp;amp;amp;lt;span style=&amp;quot;font-variant:small-caps;&amp;quot;&amp;gt;PCR&amp;lt;/span&amp;gt; primers for mitochondrial cytochrome <i>c</i> oxidase subunit <span style=&quot;font-variant:small-caps;&quot;>I</span> for marine invertebrates and application in all-taxa biotic surveys;,citation_abstract=DNA barcoding is a powerful tool for species detection, identification and discovery. Metazoan DNA barcoding is primarily based upon a specific region of the cytochrome c oxidase subunit I gene that is PCR amplified by primers HCO2198 and LCO1490 (“Folmer primers”) designed by Folmer et al. (Molecular Marine Biology and Biotechnology, 3, 1994, 294). Analysis of sequences published since 1994 has revealed mismatches in the Folmer primers to many metazoans. These sequences also show that an extremely high level of degeneracy would be necessary in updated Folmer primers to maintain broad taxonomic utility. In primers jgHCO2198 and jgLCO1490, we replaced most fully degenerated sites with inosine nucleotides that complement all four natural nucleotides and modified other sites to better match major marine invertebrate groups. The modified primers were used to amplify and sequence cytochrome c oxidase subunit I from 9105 specimens from Moorea, French Polynesia and San Francisco Bay, California, USA representing 23 phyla, 42 classes and 121 orders. The new primers, jgHCO2198 and jgLCO1490, are well suited for routine DNA barcoding, all-taxon surveys and metazoan metagenomics.;,citation_author=J. Geller;,citation_author=C. Meyer;,citation_author=M. Parker;,citation_author=H. Hawk;,citation_publication_date=2013-09;,citation_cover_date=2013-09;,citation_year=2013;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12138;,citation_issue=5;,citation_doi=10.1111/1755-0998.12138;,citation_issn=1755-098X, 1755-0998;,citation_volume=13;,citation_language=en-US;,citation_journal_title=Molecular Ecology Resources;">
<meta name="citation_reference" content="citation_title=Regression and Other Stories (corrections up to 26 Oct 2023) Please do not reproduce in any form without permission;,citation_author=Andrew Gelman;,citation_author=Jennifer Hill;,citation_author=Aki Vehtari;,citation_publication_date=2024-07;,citation_cover_date=2024-07;,citation_year=2024;,citation_fulltext_html_url=https://avehtari.github.io/ROS-Examples/;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Genome-scale community modelling reveals conserved metabolic cross-feedings in epipelagic bacterioplankton communities;,citation_abstract=Abstract Marine microorganisms form complex communities of interacting organisms that influence central ecosystem functions in the ocean such as primary production and nutrient cycling. Identifying the mechanisms controlling their assembly and activities is a major challenge in microbial ecology. Here, we integrated Tara Oceans meta-omics data to predict genome-scale community interactions within prokaryotic assemblages in the euphotic ocean. A global genome-resolved co-activity network revealed a significant number of inter-lineage associations across diverse phylogenetic distances. Identified co-active communities include species displaying smaller genomes but encoding a higher potential for quorum sensing, biofilm formation, and secondary metabolism. Community metabolic modelling reveals a higher potential for interaction within co-active communities and points towards conserved metabolic cross-feedings, in particular of specific amino acids and group B vitamins. Our integrated ecological and metabolic modelling approach suggests that genome streamlining and metabolic auxotrophies may act as joint mechanisms shaping bacterioplankton community assembly in the global ocean surface.;,citation_author=Nils Giordano;,citation_author=Marinna Gaudin;,citation_author=Camille Trottier;,citation_author=Erwan Delage;,citation_author=Charlotte Nef;,citation_author=Chris Bowler;,citation_author=Samuel Chaffron;,citation_publication_date=2024-03;,citation_cover_date=2024-03;,citation_year=2024;,citation_fulltext_html_url=https://www.nature.com/articles/s41467-024-46374-w;,citation_issue=1;,citation_doi=10.1038/s41467-024-46374-w;,citation_issn=2041-1723;,citation_volume=15;,citation_language=en-US;,citation_journal_title=Nat Commun;">
<meta name="citation_reference" content="citation_title=Ecological Sexual Dimorphism and Environmental Variability within a Community of Antarctic Penguins (Genus Pygoscelis);,citation_abstract=Background: Sexual segregation in vertebrate foraging niche is often associated with sexual size dimorphism (SSD), i.e., ecological sexual dimorphism. Although foraging behavior of male and female seabirds can vary markedly, differences in isotopic (carbon, d13C and nitrogen, d15N) foraging niche are generally more pronounced within sexually dimorphic species and during phases when competition for food is greater. We examined ecological sexual dimorphism among sympatric nesting Pygoscelis penguins asking whether environmental variability is associated with differences in male and female prebreeding foraging niche. We predicted that all Pygoscelis species would forage sex-specifically, and that higher quality winter habitat, i.e., higher or lower sea ice coverage for a given species, would be associated with a more similar foraging niche among the sexes. Results: P2/P8 primers reliably amplified DNA of all species. On average, male Pygoscelis penguins are structurally larger than female conspecifics. However, chinstrap penguins were more sexually dimorphic in culmen and flipper features than Adelie and gentoo penguins. Adelies and gentoos were more sexually dimorphic in body mass than chinstraps. Only male and female chinstraps and gentoos occupied separate d15N foraging niches. Strong year effects in d15N signatures were documented for all three species, however, only for Adelies, did yearly variation in d15N signatures tightly correlate with winter sea ice conditions. There was no evidence that variation in sex-specific foraging niche interacted with yearly winter habitat quality. Conclusion: Chinstraps were most sexually size dimorphic followed by gentoos and Adelies. Pre-breeding sex-specific foraging niche was associated with overall SSD indices across species; male chinstrap and gentoo penguins were enriched in d15N relative to females. Our results highlight previously unknown trophic pathways that link Pygoscelis penguins with variation in Southern Ocean sea ice suggesting that each sex within a species should respond similarly in pre-breeding trophic foraging to changes in future winter habitat.;,citation_author=Kristen B Gorman;,citation_author=Tony D Williams;,citation_author=William R Fraser;,citation_publication_date=2014;,citation_cover_date=2014;,citation_year=2014;,citation_issue=3;,citation_doi=10.1371/journal.pone.0090081;,citation_volume=9;,citation_language=en-US;,citation_journal_title=PLOS ONE;">
<meta name="citation_reference" content="citation_title=Traité de zoologie: anatomie, systématique, biologie;,citation_author=Pierre-Paul Grassé;,citation_author=Jacques Forest;,citation_publication_date=1996;,citation_cover_date=1996;,citation_year=1996;,citation_isbn=978-2-225-84973-2;,citation_language=fre;,citation_series_title=Tome VII, Fascicule 2, Crustacés.;">
<meta name="citation_reference" content="citation_title=Metapopulation dynamics;,citation_author=Ilkka Hanski;,citation_publication_date=1998-11;,citation_cover_date=1998-11;,citation_year=1998;,citation_fulltext_html_url=https://www.nature.com/articles/23876;,citation_issue=6706;,citation_doi=10.1038/23876;,citation_issn=0028-0836, 1476-4687;,citation_volume=396;,citation_language=en-US;,citation_journal_title=Nature;">
<meta name="citation_reference" content="citation_title=Comparing isotopic niche widths among and within communities: SIBER - Stable Isotope Bayesian Ellipses in R: Bayesian isotopic niche metrics;,citation_author=Andrew L. Jackson;,citation_author=Richard Inger;,citation_author=Andrew C. Parnell;,citation_author=Stuart Bearhop;,citation_publication_date=2011-05;,citation_cover_date=2011-05;,citation_year=2011;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2656.2011.01806.x;,citation_issue=3;,citation_doi=10.1111/j.1365-2656.2011.01806.x;,citation_issn=00218790;,citation_volume=80;,citation_language=en-US;,citation_journal_title=Journal of Animal Ecology;">
<meta name="citation_reference" content="citation_title=Guide to the marine isopod crustaceans of the Caribbean;,citation_author=Brain Frederick Kensley;,citation_author=Marilyn Schotte;,citation_publication_date=1989;,citation_cover_date=1989;,citation_year=1989;,citation_isbn=0-87474-724-4;,citation_language=en-US;">
<meta name="citation_reference" content="citation_title=Microsatellite Data Analysis for Population Genetics;,citation_abstract=Theories and analytical tools of population genetics have been widely applied for addressing various questions in the fields of ecological genetics, conservation biology, and any context where the role of dispersal or gene flow is important. Underlying much of population genetics is the analysis of variation at selectively neutral marker loci, and microsatellites continue to be a popular choice of marker. In recent decades, software programs to estimate population genetics parameters have been developed at an increasing pace as computational science and theoretical knowledge advance. Numerous population genetics software programs are presently available to analyze microsatellite genotype data, but only a handful are commonly employed for calculating parameters such as genetic variation, genetic structure, patterns of spatial and temporal gene flow, population demography, individual population assignment, and genetic relationships within and between populations. In this chapter, we introduce statistical analyses and relevant population genetic software programs that are commonly employed in the field of population genetics and molecular ecology.;,citation_author=Kyung Seok Kim;,citation_author=Thomas W. Sappington;,citation_editor=Stella K. Kantartzi;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_fulltext_html_url=https://link.springer.com/10.1007/978-1-62703-389-3_19;,citation_doi=10.1007/978-1-62703-389-3_19;,citation_isbn=978-1-62703-388-6 978-1-62703-389-3;,citation_volume=1006;,citation_language=en-US;,citation_inbook_title=Microsatellites;">
<meta name="citation_reference" content="citation_title=Correct Species Identification and Its Implications for Conservation Using Haploniscidae (Crustacea, Isopoda) in Icelandic Waters as a Proxy;,citation_abstract=Correct identification of species is required to assess and understand the biodiversity of an ecosystem. In the deep sea, however, this is only possible to a limited extent, as a large part of the fauna is undescribed and the identification keys for most taxa are inadequate or missing. With the progressive impact of climate change and anthropogenic activities on deep-sea ecosystems, it is imperative to define reliable methods for robust species identification. In this study, different techniques for the identification of deep-sea species are tested, including a combination of morphological, molecular (DNA barcoding, and proteomic fingerprinting), biogeographical and ecological modeling approaches. These are applied to a family of isopods, the Haploniscidae, from deep waters around Iceland. The construction of interactive identification keys based on the DELTA format (DEscription Language for TAxonomy) were a major pillar of this study, the evaluation of which was underpinned by the application of the supplementary methods. Overall, interactive keys have been very reliable in identifying species within the Haploniscidae. Especially in a deep-sea context, these types of keys could become established because they are easy to adapt and flexible enough to accommodate newly described species. Remarkably, in this study, the interactive key enabled identification of a supposedly new species within the Haploniscidae that was later verified using both molecular genetic – and proteomic methods. However, these keys are limited given that they are based on purely morphological characteristics, including where species with strong ontogenetic or sexual dimorphism occur as both genders are not always described. In this case, integrative taxonomy is the method of choice and the combination presented here has been shown to be very promising for correct identification of deep-sea isopods.;,citation_author=Karlotta Kürzel;,citation_author=Stefanie Kaiser;,citation_author=Anne-Nina Lörz;,citation_author=Sven Rossel;,citation_author=Eva Paulus;,citation_author=Janna Peters;,citation_author=Martin Schwentner;,citation_author=Pedro Martinez Arbizu;,citation_author=Charles Oliver Coleman;,citation_author=Jörundur Svavarsson;,citation_author=Saskia Brix;,citation_publication_date=2022-01;,citation_cover_date=2022-01;,citation_year=2022;,citation_fulltext_html_url=https://www.frontiersin.org/articles/10.3389/fmars.2021.795196/full;,citation_doi=10.3389/fmars.2021.795196;,citation_issn=2296-7745;,citation_volume=8;,citation_language=en-US;,citation_journal_title=Front. Mar. Sci.;">
<meta name="citation_reference" content="citation_title=Acanthocope;,citation_author=undefined Kussakin;,citation_publication_date=2003;,citation_cover_date=2003;,citation_year=2003;,citation_inbook_title=Book Section: Acanthocope;">
<meta name="citation_reference" content="citation_title=Identification and characterization of microsatellites from the Antarctic isopod Ceratoserolis trilobitoides: Nuclear evidence for cryptic species;,citation_author=Florian Leese;,citation_author=Christoph Held;,citation_publication_date=2008-10;,citation_cover_date=2008-10;,citation_year=2008;,citation_fulltext_html_url=http://link.springer.com/10.1007/s10592-007-9491-z;,citation_issue=5;,citation_doi=10.1007/s10592-007-9491-z;,citation_issn=1566-0621, 1572-9737;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Conserv Genet;">
<meta name="citation_reference" content="citation_title=Structural size measurements and isotopic signatures of foraging among adult male and female Chinstrap penguins (Pygoscelis antarcticus) nesting along the Palmer Archipelago near Palmer Station, 2007-2009;,citation_abstract=Sexual segregation in vertebrate foraging niche is often associated with sexual size dimorphism (SSD), i.e., ecological sexual dimorphism. We examined ecological sexual dimorphism among sympatric nesting Pygoscelis penguins near Palmer Station, Antarctica, asking whether environmental variability in the form of winter sea ice is associated with differences in male and female pre-breeding foraging niche. Each season, study nests, where pairs of adults were present, were individually marked and chosen before the onset of egg-laying, and consistently monitored. When study nests were found at the one-egg stage, both adults were captured to obtain blood samples used for molecular sexing and stable isotope analyses, and measurements of structural size and body mass. At the time of capture, each adult penguin was quickly blood sampled (~1 ml) from the brachial vein. After handling, individuals at study nests were further monitored to ensure the pair reached clutch completion, i.e., two eggs. Molecular analyses were conducted at Simon Fraser University following standard PCR protocols, and stable isotope analyses were conducted at the Stable Isotope Facility at the University of California, Davis using an elemental analyzer interfaced with an isotope ratio mass spectrometer;,citation_author=Palmer Station Antarctica LTER;,citation_author=Kristen Gorman;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://portal.edirepository.org/nis/mapbrowse?packageid=knb-lter-pal.221.8;,citation_doi=10.6073/PASTA/CE9B4713BB8C065A8FCFD7F50BF30DDE;,citation_language=en-US;,citation_publisher=Environmental Data Initiative;">
<meta name="citation_reference" content="citation_title=Structural size measurements and isotopic signatures of foraging among adult male and female gentoo penguins (Pygoscelis papua) nesting along the Palmer Archipelago near Palmer Station, 2007-2009;,citation_abstract=Sexual segregation in vertebrate foraging niche is often associated with sexual size dimorphism (SSD), i.e., ecological sexual dimorphism. We examined ecological sexual dimorphism among sympatric nesting Pygoscelis penguins near Palmer Station, Antarctica, asking whether environmental variability in the form of winter sea ice is associated with differences in male and female pre-breeding foraging niche. Each season, study nests, where pairs of adults were present, were individually marked and chosen before the onset of egg-laying, and consistently monitored. When study nests were found at the one-egg stage, both adults were captured to obtain blood samples used for molecular sexing and stable isotope analyses, and measurements of structural size and body mass. At the time of capture, each adult penguin was quickly blood sampled (~1 ml) from the brachial vein. After handling, individuals at study nests were further monitored to ensure the pair reached clutch completion, i.e., two eggs. Molecular analyses were conducted at Simon Fraser University following standard PCR protocols, and stable isotope analyses were conducted at the Stable Isotope Facility at the University of California, Davis using an elemental analyzer interfaced with an isotope ratio mass spectrometer;,citation_author=Palmer Station Antarctica LTER;,citation_author=Kristen Gorman;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://portal.edirepository.org/nis/mapbrowse?packageid=knb-lter-pal.220.7;,citation_doi=10.6073/PASTA/9FC8F9B5A2FA28BDCA96516649B6599B;,citation_language=en-US;,citation_publisher=Environmental Data Initiative;">
<meta name="citation_reference" content="citation_title=Structural size measurements and isotopic signatures of foraging among adult male and female Adélie penguins (Pygoscelis adeliae) nesting along the Palmer Archipelago near Palmer Station, 2007-2009;,citation_abstract=Sexual segregation in vertebrate foraging niche is often associated with sexual size dimorphism (SSD), i.e., ecological sexual dimorphism. We examined ecological sexual dimorphism among sympatric nesting Pygoscelis penguins near Palmer Station, Antarctica, asking whether environmental variability in the form of winter sea ice is associated with differences in male and female pre-breeding foraging niche. Each season, study nests, where pairs of adults were present, were individually marked and chosen before the onset of egg-laying, and consistently monitored. When study nests were found at the one-egg stage, both adults were captured to obtain blood samples used for molecular sexing and stable isotope analyses, and measurements of structural size and body mass. At the time of capture, each adult penguin was quickly blood sampled (~1 ml) from the brachial vein. After handling, individuals at study nests were further monitored to ensure the pair reached clutch completion, i.e., two eggs. Molecular analyses were conducted at Simon Fraser University following standard PCR protocols, and stable isotope analyses were conducted at the Stable Isotope Facility at the University of California, Davis using an elemental analyzer interfaced with an isotope ratio mass spectrometer;,citation_author=Palmer Station Antarctica LTER;,citation_author=Kristen Gorman;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_fulltext_html_url=https://portal.edirepository.org/nis/mapbrowse?packageid=knb-lter-pal.219.5;,citation_doi=10.6073/PASTA/98B16D7D563F265CB52372C8CA99E60F;,citation_language=en-US;,citation_publisher=Environmental Data Initiative;">
<meta name="citation_reference" content="citation_title=A new obligate groundwater species of Asellus (Isopoda, Asellidae) from Iran;,citation_abstract=With only 43 described stygobionts and only two isopod species the obligate groundwater fauna of Iran, a vast country with over 10% of limestone surface, is inadequately known. Here, we report the discovery of Asellus ismailsezarii sp. nov. from Zagros mountains, the first eyeless and depigmented asellid isopod from Iran. The new species is morphologically similar to Asellus monticola Birstein, 1932, but it is eyeless and fully depigmented, has a slightly curved pereopod IV and does not bear any setae on proximal margins of exopodite of pleopods IV and V. Species phylogenetic relationships using original and datamined mitochondrial DNA and nuclear rDNA, and estimation of molecular divergences with other Asellus species, suggest that A. ismailsezarii sp. nov. is sister to a larger clade that also contains the European A. aquaticus species complex. Surface populations of Asellus have colonized groundwater at multiple occasions and localities, both in Europe and Asia, giving rise to species and subspecies that have evolved troglomorphisms, such as depigmentation and loss of eyes. Of the 37 formally described species and subspecies of Asellus, 15 are from groundwater, including A. ismailsezarii sp. nov. We predict that many more obligate groundwater Asellus taxa are yet to be discovered in Asia.;,citation_author=Mohammad Javad Malek-Hosseini;,citation_author=Jure Jugovic;,citation_author=Yaser Fatemi;,citation_author=Matjaž Kuntner;,citation_author=Rok Kostanjšek;,citation_author=Christophe J. Douady;,citation_author=Florian Malard;,citation_publication_date=2022-03;,citation_cover_date=2022-03;,citation_year=2022;,citation_fulltext_html_url=https://subtbiol.pensoft.net/article/79447/;,citation_doi=10.3897/subtbiol.42.79447;,citation_issn=1314-2615, 1768-1448;,citation_volume=42;,citation_language=en-US;,citation_journal_title=SB;">
<meta name="citation_reference" content="citation_title=Acanthocope mendeleevi: A new species of Munnopsidae (Crustacea, Isopoda, Asellota) from the New Caledonian Basin;,citation_author=Marina Malyutina;,citation_publication_date=1998-09;,citation_cover_date=1998-09;,citation_year=1998;,citation_issue=5;,citation_volume=24;,citation_language=en-US;,citation_journal_title=Russian Journal of Marine Biology;">
<meta name="citation_reference" content="citation_title=A new record of species of Acanthocope Isopoda, Munnopsidae);,citation_author=Marina Malyutina;,citation_publication_date=1999-07;,citation_cover_date=1999-07;,citation_year=1999;,citation_issue=4;,citation_volume=25;,citation_journal_title=Russian Journal of Marine Biology;">
<meta name="citation_reference" content="citation_title=Revision of StorthynguraVanhöffen, 1914 (Crustacea: Isopoda: Munnopsididae) with descriptions of three new genera and four new species from the deep South Atlantic;,citation_abstract=The deep-sea genus Storthyngura (family Munnopsididae) is revised. Three new genera (Rectisura,Sursumura and Vanhoeffenella) and four new species (Rectisurarichardsoniae, Sursumuraaberrata, Vanhoeffenellageorgei, and V.moskalevi) are described from deep-sea basins and trenches of the South Atlantic Ocean. Thirty-three previously described species are placed in new combinations with one of the new genera. Diagnoses are presented for the subfamily Storthyngurinae Kussakin (2003), the genus StorthynguraVanhöffen, and all new taxa. The accompanying Electronic Supplement offers a key to the six genera in the subfamily, and keys to species in Storthynguraand the three new genera.;,citation_author=Marina V Malyutina;,citation_publication_date=2003;,citation_cover_date=2003;,citation_year=2003;,citation_language=en-US;,citation_journal_title=Org. Divers. Evol.;">
<meta name="citation_reference" content="citation_title=Acanthocopinae (Crustacea: Isopoda: Munnopsididae) from the Southern Ocean deep sea with the description of Acanthocope eleganta sp. nov.;,citation_abstract=Acanthocope eleganta sp. nov. is described from the abyssal Southern Ocean near the Southern Ocean Peninsula. The new species differs from others in the following: a slender dorsomedial spine on the pleon anteriorly, a pair of short dorsal spines and long ventral spine on each of pereonites 5 and 6; uropods half as long as the terminal spine of the pleotelson and with a minute exopod. A. annulatus Menzies, 1962 is redescribed; A. galatheae Wolff, 1962, previously known only from the Gulf of Panama and from Angola Basin, is recorded from the northwest Weddell Sea.;,citation_author=Marina Malyutina;,citation_author=Angelika Brandt;,citation_publication_date=2004-06;,citation_cover_date=2004-06;,citation_year=2004;,citation_fulltext_html_url=https://mapress.com/zt/article/view/zootaxa.550.1.1;,citation_issue=1;,citation_doi=10.11646/zootaxa.550.1.1;,citation_issn=1175-5334, 1175-5326;,citation_volume=550;,citation_language=en-US;,citation_journal_title=Zootaxa;">
<meta name="citation_reference" content="citation_title=Diversity and distribution of the deep-sea Atlantic Acanthocope (Crustacea, Isopoda, Munnopsidae), with description of two new species;,citation_abstract=The present paper provides data of the world-wide distribution, abundance and diversity of the rare deep-sea munnopsid genus, Acanthocope Beddard, 1885 and presents an identification key to the 18 known species of Acanthocope. Two new species, Acanthocope puertoricana sp. nov. from the tropical abyssal area near the Puerto Rico Trench and Acanthocope galaica sp. nov. from the Galicia Bank, Northeastern Atlantic are described. Additionally, Acanthocope annulatus Menzies, 1962 and Acanthocope eleganta Malyutina and Brandt, 2004 have been collected in the Puerto Rico Trench area, whereas Acanthocope galatheae Wolff, 1962 were collected along a latitudinal transect crossing the tropical abyssal North Atlantic during the Vema-TRANSIT expedition. A. puertoricana is most similar to A. carinata Chardy, 1972 from the Northwestern Atlantic and differs from the latter by: a more slender body; a broader head; a more slender and shorter lateral projections of the pleotelson; the presence of ventral spines on natasome; and a longer uropod. A. galaica sp. nov. is most similar to A. spinicauda Beddard, 1885 and differs from the latter by having a more elongate pleotelson with almost straight lateral margins between lateral spines, whereas the margins are more convex in A. spinicauda. Pereopods 5 and 6 of A. galaica have broader carpi than in A. spinicauda.;,citation_author=Marina V. Malyutina;,citation_author=Inmaculada Frutos;,citation_author=Angelika Brandt;,citation_publication_date=2018-02;,citation_cover_date=2018-02;,citation_year=2018;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0967064517300280;,citation_doi=10.1016/j.dsr2.2017.11.003;,citation_issn=09670645;,citation_volume=148;,citation_language=en-US;,citation_journal_title=Deep Sea Research Part II: Topical Studies in Oceanography;">
<meta name="citation_reference" content="citation_title=“Deep-Sea Spiders” That Walk Through the Water;,citation_abstract=ABSTRACT Deep-sea isopods of the family Munnopsidae exhibit four modes of swimming: forward striding, slow backward pedalling, fast backward pedalling and escape, the first two of which use asymmetric phases of leg movement. Instead of moving the left and right limbs (pereopods, P2–P7) of a segment in-phase (e.g. RP2LP2, RP3LP3, RP4LP4), as do most aquatic insects, leg movement is more like that of fast-walking insects, where closest synchrony occurs between diagonal limbs (e.g. RP2LP3, RP3LP4, RP4LP2). This is similar to the alternating tripod gait used by many animals on land to prevent them from toppling over. It therefore seems likely that this group of isopods learned to walk before they could swim.;,citation_author=N. Justin Marshall;,citation_author=Carol Diebel;,citation_publication_date=1995-06;,citation_cover_date=1995-06;,citation_year=1995;,citation_fulltext_html_url=https://journals.biologists.com/jeb/article/198/6/1371/6961/Deep-Sea-Spiders-That-Walk-Through-the-Water;,citation_issue=6;,citation_doi=10.1242/jeb.198.6.1371;,citation_issn=0022-0949, 1477-9145;,citation_volume=198;,citation_language=en-US;,citation_journal_title=Journal of Experimental Biology;">
<meta name="citation_reference" content="citation_title=New bathyal Isopoda from the Caribbean with observations on their nutrition;,citation_author=Robert J. Menzies;,citation_publication_date=1956;,citation_cover_date=1956;,citation_year=1956;,citation_journal_title=Museum of comparative zoology;,citation_series_title=63th series;">
<meta name="citation_reference" content="citation_title=The Isopods of Abyssal Depth in the Atlantic Ocean;,citation_author=Robert J. Menzies;,citation_publication_date=1962;,citation_cover_date=1962;,citation_year=1962;">
<meta name="citation_reference" content="citation_title=Stable isotopes as descriptors of ecological niches;,citation_abstract=Part of the course &amp;amp;amp;quot;Applications of stable isotopes in marine ecosystems&amp;quot;, taught by Gilles Lepoint &amp;amp; Loïc Michel at University of Liège (Belgium).;,citation_author=Loïc N. Michel;,citation_author=Gilles Lepoint;,citation_publication_date=2021-12;,citation_cover_date=2021-12;,citation_year=2021;,citation_fulltext_html_url=https://zenodo.org/record/5765530;,citation_doi=10.5281/ZENODO.5765530;,citation_language=en-US;,citation_publisher=Zenodo;">
<meta name="citation_reference" content="citation_title=Stable isotope analyses identify trophic niche partitioning between sympatric terrestrial vertebrates in coastal saltmarshes with differing oiling histories;,citation_abstract=Bioindicator species are commonly used as proxies to help identify the ecological effects of oil spills and other stressors. However, the utility of taxa as bioindicators is dependent on understanding their trophic niche and life history characteristics, as these factors mediate their ecological responses. Seaside sparrows ( Ammospiza maritima) and marsh rice rats ( Oryzomys palustris ) are two ubiquitous terrestrial vertebrates that are thought to be bioindicators of oil spills in saltmarsh ecosystems. To improve the utility of these omnivorous taxa as bioindicators, we used carbon and nitrogen stable isotope analysis to quantify their trophic niches at saltmarshes in coastal Louisiana with differing oiling histories. We found that rats generally had lower trophic positions and incorporated more aquatic prey relative to seaside sparrows. The range of resources used (i.e.,trophic niche width) varied based on oiling history. Seaside sparrows had wider trophic niches than marsh rice rats at unoiled sites, but not at oiled sites. Trophic niche widths of conspecifics were less consistent at oiled sites, although marsh rice rats at oiled sites had wider trophic niches than rats at unoiled sites. These results suggest that past oiling histories may have imparted subtle, yet differing effects on the foraging ecology of these two co-occurring species. However, the temporal lag between initial oiling and our study makes identifying the ultimate drivers of differences between oiled and unoiled sites challenging. Even so, our findings provide a baseline quantification of the trophic niches of sympatric seaside sparrows and marsh rice rats that will aid in the use of these species as indicators of oiling and other environmental stressors in saltmarsh ecosystems.;,citation_author=Sydney Moyo;,citation_author=Hayat Bennadji;,citation_author=Danielle Laguaite;,citation_author=Anna A. Pérez-Umphrey;,citation_author=Allison M. Snider;,citation_author=Andrea Bonisoli-Alquati;,citation_author=Jill A. Olin;,citation_author=Philip C Stouffer;,citation_author=Sabrina S. Taylor;,citation_author=Paola C. López-Duarte;,citation_author=Brian J. Roberts;,citation_author=Linda Hooper-Bui;,citation_author=Michael J. Polito;,citation_publication_date=2021-07;,citation_cover_date=2021-07;,citation_year=2021;,citation_fulltext_html_url=https://peerj.com/articles/11392;,citation_doi=10.7717/peerj.11392;,citation_issn=2167-8359;,citation_volume=9;,citation_language=en-US;,citation_journal_title=PeerJ;">
<meta name="citation_reference" content="citation_title=Acanthocope pentacornis n.sp. From the Deep Sea of the Gulf of Aden (Crustaceae: Isopoda: Munnopsidae);,citation_author=Hans-Georg Mueller;,citation_publication_date=1989-10;,citation_cover_date=1989-10;,citation_year=1989;,citation_volume=20;,citation_journal_title=Senckenbergiana Maritima;">
<meta name="citation_reference" content="citation_title=A biologist’s guide to Bayesian phylogenetic analysis;,citation_author=Fabrícia F. Nascimento;,citation_author=Mario Dos Reis;,citation_author=Ziheng Yang;,citation_publication_date=2017-09;,citation_cover_date=2017-09;,citation_year=2017;,citation_fulltext_html_url=https://www.nature.com/articles/s41559-017-0280-x;,citation_issue=10;,citation_doi=10.1038/s41559-017-0280-x;,citation_issn=2397-334X;,citation_volume=1;,citation_language=en-US;,citation_journal_title=Nat Ecol Evol;">
<meta name="citation_reference" content="citation_title=A niche for isotopic ecology;,citation_author=Seth D. Newsome;,citation_author=Carlos Martinez Del Rio;,citation_author=Stuart Bearhop;,citation_author=Donald L. Phillips;,citation_publication_date=2007-10;,citation_cover_date=2007-10;,citation_year=2007;,citation_fulltext_html_url=http://doi.wiley.com/10.1890/060150.1;,citation_issue=8;,citation_doi=10.1890/060150.1;,citation_issn=1540-9295;,citation_volume=5;,citation_language=en-US;,citation_journal_title=Frontiers in Ecology and the Environment;">
<meta name="citation_reference" content="citation_title=DNA barcoding the medusozoa and ctenophora;,citation_author=Brian Daniel Ortman;,citation_publication_date=2008;,citation_cover_date=2008;,citation_year=2008;">
<meta name="citation_reference" content="citation_title=Relationships within the Munnopsidae (Crustacea, Isopoda, Asellota) based on three genes;,citation_abstract=The Munnopsidae are a diverse group of asellote isopods that are an important component of deep-sea fauna. Morphologically-based phylogenetic inference attempts have proven to be of limited use due to the ecological and morphological diversity within the clade. Monophyly of the family is well-established but relationships within the group remain unresolved. This project is the first molecularly-based effort focused specifically on resolving phylogenetic relationships within the Munnopsidae. Partial 28S and COI and complete 18S genes were sequenced for 28 asellotes, 15 additional taxa were included from which only one or two of the three target sequences could be obtained, and 18S sequences for five additional taxa were available from GenBank. Sequences were analysed both as individual genes and in combination using Bayesian and maximum parsimony approaches. Each gene provided a phylogenetic signal that could be identified in the combined analyses, with 18S analyses providing the most resolution of phylogenetic relationships. The available representatives of subfamilies Munnopsinae and Ilyarachninae were monophyletic, as was the genus Munneurycope. Relationships within the subfamily Munnopsinae were well-resolved by thorough taxon sampling, several new species were placed, and the need for taxonomic revision of Munnopsis/Munnopsoides was supported. These analyses supported putative Eurycope paraphyly and emphasized the need for careful revision of this highly variable genus. Tytthocope was sister to Munnopsurus . Syneurycope was suggested as the sister group to the ilyarachnines. Combined analyses provided increased support for clades suggested in at least two individual gene analyses and for clades not strongly contradicted by individual analyses. Further work is required to fully resolve the munnopsid phylogeny and should consist of increased taxon sampling for the complete 18S sequence and possibly identification of at least one slowly evolving, nuclear protein-coding gene to resolve the basal polytomy and enable placement of the root.;,citation_author=Karen J. Osborn;,citation_publication_date=2009-11;,citation_cover_date=2009-11;,citation_year=2009;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/j.1463-6409.2009.00394.x;,citation_issue=6;,citation_doi=10.1111/j.1463-6409.2009.00394.x;,citation_issn=0300-3256, 1463-6409;,citation_volume=38;,citation_language=en-US;,citation_journal_title=Zoologica Scripta;">
<meta name="citation_reference" content="citation_title=A Systematic Review of Phenotypic Plasticity in Marine Invertebrate and Plant Systems;,citation_author=Dianna K. Padilla;,citation_author=Monique M. Savedo;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/B9780124104983000021;,citation_doi=10.1016/B978-0-12-410498-3.00002-1;,citation_isbn=978-0-12-410498-3;,citation_volume=65;,citation_language=en-US;,citation_inbook_title=Advances in Marine Biology;">
<meta name="citation_reference" content="citation_title=Assessing the efficiency of multiple sequence alignment programs;,citation_abstract=Background: Multiple sequence alignment (MSA) is an extremely useful tool for molecular and evolutionary biology and there are several programs and algorithms available for this purpose. Although previous studies have compared the alignment accuracy of different MSA programs, their computational time and memory usage have not been systematically evaluated. Given the unprecedented amount of data produced by next generation deep sequencing platforms, and increasing demand for large-scale data analysis, it is imperative to optimize the application of software. Therefore, a balance between alignment accuracy and computational cost has become a critical indicator of the most suitable MSA program. We compared both accuracy and cost of nine popular MSA programs, namely CLUSTALW, CLUSTAL OMEGA, DIALIGN-TX, MAFFT, MUSCLE, POA, Probalign, Probcons and T-Coffee, against the benchmark alignment dataset BAliBASE and discuss the relevance of some implementations embedded in each program’s algorithm. Accuracy of alignment was calculated with the two standard scoring functions provided by BAliBASE, the sum-of-pairs and total-column scores, and computational costs were determined by collecting peak memory usage and time of execution. Results: Our results indicate that mostly the consistency-based programs Probcons, T-Coffee, Probalign and MAFFT outperformed the other programs in accuracy. Whenever sequences with large N/C terminal extensions were present in the BAliBASE suite, Probalign, MAFFT and also CLUSTAL OMEGA outperformed Probcons and T-Coffee. The drawback of these programs is that they are more memory-greedy and slower than POA, CLUSTALW, DIALIGN-TX, and MUSCLE. CLUSTALW and MUSCLE were the fastest programs, being CLUSTALW the least RAM memory demanding program. Conclusions: Based on the results presented herein, all four programs Probcons, T-Coffee, Probalign and MAFFT are well recommended for better accuracy of multiple sequence alignments. T-Coffee and recent versions of MAFFT can deliver faster and reliable alignments, which are specially suited for larger datasets than those encountered in the BAliBASE suite, if multi-core computers are available. In fact, parallelization of alignments for multi-core computers should probably be addressed by more programs in a near future, which will certainly improve performance significantly.;,citation_author=Fabiano Sviatopolk-Mirsky Pais;,citation_author=Patrícia De Cássia Ruy;,citation_author=Guilherme Oliveira;,citation_author=Roney Santos Coimbra;,citation_publication_date=2014-12;,citation_cover_date=2014-12;,citation_year=2014;,citation_fulltext_html_url=https://almob.biomedcentral.com/articles/10.1186/1748-7188-9-4;,citation_issue=1;,citation_doi=10.1186/1748-7188-9-4;,citation_issn=1748-7188;,citation_volume=9;,citation_language=en-US;,citation_journal_title=Algorithms Mol Biol;">
<meta name="citation_reference" content="citation_title=Dominance of Wolbachia sp. In the deep-sea sediment bacterial metataxonomic sequencing analysis in the Bay of Bengal, Indian Ocean;,citation_author=A. Parvathi;,citation_author=V. Jasna;,citation_author=V. K. Aswathy;,citation_author=S. Aparna;,citation_author=V. K. Nathan;,citation_author=R. Jyothibabu;,citation_publication_date=2020-01;,citation_cover_date=2020-01;,citation_year=2020;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S0888754318306761;,citation_issue=1;,citation_doi=10.1016/j.ygeno.2019.06.019;,citation_issn=08887543;,citation_volume=112;,citation_language=en-US;,citation_journal_title=Genomics;">
<meta name="citation_reference" content="citation_title=Recent speciation and hybridization in Icelandic deep-sea isopods: An integrative approach using genomics and proteomics;,citation_abstract=The crustacean marine isopod species Haploniscus bicuspis (Sars, 1877) shows circumIcelandic distribution in a wide range of environmental conditions and along wellknown geographic barriers, such as the Greenland-Iceland-Faroe (GIF) Ridge. We wanted to explore population genetics, phylogeography and cryptic speciation as well as investigate whether previously described, but unaccepted subspecies have any merit. Using the same set of specimens, we combined mitochondrial COI sequences, thousands of nuclear loci (ddRAD), and proteomic profiles, plus selected morphological characters using confocal laser scanning microscopy (CLSM). Five divergent genetic lineages were identified by COI and ddRAD, two south and three north of the GIF Ridge. Assignment of populations to the three northern lineages varied and detailed analyses revealed hybridization and gene flow between them, suggesting a single northern species with a complex phylogeographic history. No apparent hybridization was observed among lineages south of the GIF Ridge, inferring the existence of two more species. Differences in proteomic profiles between the three putative species were minimal, implying an ongoing or recent speciation process. Population differentiation was high, even among closely associated populations, and higher in m-i tochondrial COI than nuclear ddRAD loci. Gene flow is apparently male- biased, leading to hybrid zones and instances of complete exchange of the local nuclear genome through immigrating males. This study did not confirm the existence of subspecies defined by male characters, which probably instead refer to different male developmental stages.;,citation_author=Eva Paulus;,citation_author=Saskia Brix;,citation_author=Annabelle Siebert;,citation_author=Pedro Martínez Arbizu;,citation_author=Sven Rossel;,citation_author=Janna Peters;,citation_author=Jörundur Svavarsson;,citation_author=Martin Schwentner;,citation_publication_date=2022-01;,citation_cover_date=2022-01;,citation_year=2022;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1111/mec.16234;,citation_issue=1;,citation_doi=10.1111/mec.16234;,citation_issn=0962-1083, 1365-294X;,citation_volume=31;,citation_language=en-US;,citation_journal_title=Molecular Ecology;">
<meta name="citation_reference" content="citation_title=Cryptic speciation or global spread? The case of a cosmopolitan marine invertebrate with limited dispersal capabilities;,citation_author=R. Pérez-Portela;,citation_author=V. Arranz;,citation_author=M. Rius;,citation_author=X. Turon;,citation_publication_date=2013-11;,citation_cover_date=2013-11;,citation_year=2013;,citation_fulltext_html_url=https://www.nature.com/articles/srep03197;,citation_issue=1;,citation_doi=10.1038/srep03197;,citation_issn=2045-2322;,citation_volume=3;,citation_language=en-US;,citation_journal_title=Sci Rep;">
<meta name="citation_reference" content="citation_title=NetCoMi: Network construction and comparison for microbiome data in R;,citation_abstract=Abstract Motivation Estimating microbial association networks from high-throughput sequencing data is a common exploratory data analysis approach aiming at understanding the complex interplay of microbial communities in their natural habitat. Statistical network estimation workflows comprise several analysis steps, including methods for zero handling, data normalization and computing microbial associations. Since microbial interactions are likely to change between conditions, e.g. between healthy individuals and patients, identifying network differences between groups is often an integral secondary analysis step. Thus far, however, no unifying computational tool is available that facilitates the whole analysis workflow of constructing, analysing and comparing microbial association networks from high-throughput sequencing data. Results Here, we introduce NetCoMi (Network Construction and comparison for Microbiome data), an R package that integrates existing methods for each analysis step in a single reproducible computational workflow. The package offers functionality for constructing and analysing single microbial association networks as well as quantifying network differences. This enables insights into whether single taxa, groups of taxa or the overall network structure change between groups. NetCoMi also contains functionality for constructing differential networks, thus allowing to assess whether single pairs of taxa are differentially associated between two groups. Furthermore, NetCoMi facilitates the construction and analysis of dissimilarity networks of microbiome samples, enabling a high-level graphical summary of the heterogeneity of an entire microbiome sample collection. We illustrate NetCoMi’s wide applicability using data sets from the GABRIELA study to compare microbial associations in settled dust from children’s rooms between samples from two study centers (Ulm and Munich). Availability R scripts used for producing the examples shown in this manuscript are provided as supplementary data. The NetCoMi package, together with a tutorial, is available at https://github.com/stefpeschel/NetCoMi. Contact Tel:+49 89 3187 43258; stefanie.peschel@mail.de Supplementary information Supplementary data are available at Briefings in Bioinformatics online.;,citation_author=Stefanie Peschel;,citation_author=Christian L Müller;,citation_author=Erika Von Mutius;,citation_author=Anne-Laure Boulesteix;,citation_author=Martin Depner;,citation_publication_date=2021-07;,citation_cover_date=2021-07;,citation_year=2021;,citation_fulltext_html_url=https://academic.oup.com/bib/article/doi/10.1093/bib/bbaa290/6017455;,citation_issue=4;,citation_doi=10.1093/bib/bbaa290;,citation_issn=1467-5463, 1477-4054;,citation_volume=22;,citation_language=en-US;,citation_journal_title=Briefings in Bioinformatics;">
<meta name="citation_reference" content="citation_title=Multiple origins of deep-sea Asellota (Crustacea: Isopoda) from shallow waters revealed by molecular data;,citation_abstract=The Asellota are a highly variable group of Isopoda with many species in freshwater and marine shallow-water environments. However, in the deep sea, they show their most impressive radiation with a broad range of astonishing morphological adaptations and bizarre body forms. Nevertheless, the evolution and phylogeny of the deep-sea Asellota are poorly known because of difficulties in scoring morphological characters. In this study, the molecular phylogeny of the Asellota is evaluated for 15 marine shallow-water species and 101 deep-sea species, using complete 18S and partial 28S rDNA gene sequences. Our molecular data support the monophyly of most deep-sea families and give evidence for a multiple colonization of the deep sea by at least four major lineages of asellote isopods. According to our molecular data, one of these lineages indicates an impressive radiation in the deep sea. Furthermore, the present study rejects the monophyly of the family Janiridae, a group of plesiomorphic shallow-water Asellota, and several shallow-water and deep-sea genera ( Acanthaspidia , Ianthopsis , Haploniscus , Echinozone , Eurycope , Munnopsurus and Syneurycope ).;,citation_author=Michael J Raupach;,citation_author=Christoph Mayer;,citation_author=Marina Malyutina;,citation_author=Johann-Wolfgang Wägele;,citation_publication_date=2009-03;,citation_cover_date=2009-03;,citation_year=2009;,citation_fulltext_html_url=https://royalsocietypublishing.org/doi/10.1098/rspb.2008.1063;,citation_issue=1658;,citation_doi=10.1098/rspb.2008.1063;,citation_issn=0962-8452, 1471-2954;,citation_volume=276;,citation_language=en-US;,citation_journal_title=Proc. R. Soc. B.;">
<meta name="citation_reference" content="citation_title=Geographically distinct patterns of reproductive isolation and hybridization in two sympatric species of the &amp;amp;amp;lt;i&amp;gt;Jaera&amp;lt;/i&amp;gt; <i>Albifrons</i> complex (marine isopods);,citation_abstract=Abstract Sympatric species that in some populations hybridize and in other populations remain reproductively isolated open interesting research possibilities for the study of hybridization and speciation. Here, we test for such a situation in two littoral isopods ( Jaera albifrons and J.&nbsp;praehirsuta ) that occur in mixed populations and where past morphological descriptions suggested that the two species are generally reproductively isolated except in rare populations where hybridization may be happening. Using field surveys and microsatellite genetic structure analyses in two regions from France (Normandy and Brittany), we confirmed that introgressive hybridization occurs in a subset of mixed J.&nbsp;albifrons / J.&nbsp;praehirsuta populations (region Normandy) where the two species are found in the same habitat (pebbles on the shore). Moreover, we found that introgression in these populations is differential, 21 of 23 microsatellite markers showing little genetic divergence between species (hierarchical analysis of molecular variance F CT &nbsp;=&nbsp;0.017) while the remaining two loci were strongly differentiated ( F CT &nbsp;=&nbsp;0.428). By contrast, J.&nbsp;albifrons and J.&nbsp;praehirsuta in mixed populations from region Brittany occupied distinct habitats (pebbles and seaweeds, respectively) with little overlap and showed stronger genetic divergence ( F CT &nbsp;=&nbsp;0.132). In hybridizing populations, the majority of individuals show morphological traits that are characteristic of one or the other species. This raises the question of the forces that act to maintain this polymorphism, noting that hybridizing populations seem to be geographically isolated from potential source parental populations and show no detectable habitat divergence between species.;,citation_author=Ambre Ribardière;,citation_author=Claire Daguin-Thiébaut;,citation_author=Céline Houbin;,citation_author=Jérôme Coudret;,citation_author=Caroline Broudin;,citation_author=Olivier Timsit;,citation_author=Thomas Broquet;,citation_publication_date=2017-07;,citation_cover_date=2017-07;,citation_year=2017;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1002/ece3.3106;,citation_issue=14;,citation_doi=10.1002/ece3.3106;,citation_issn=2045-7758, 2045-7758;,citation_volume=7;,citation_language=en-US;,citation_journal_title=Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=Caught in the act: An abyssal isopod collected while feeding on Komokiaceae;,citation_author=Torben Riehl;,citation_author=Simon Bober;,citation_author=Ivan Voltski;,citation_author=Marina V. Malyutina;,citation_author=Angelika Brandt;,citation_publication_date=2018-03;,citation_cover_date=2018-03;,citation_year=2018;,citation_fulltext_html_url=http://link.springer.com/10.1007/s12526-016-0606-y;,citation_issue=1;,citation_doi=10.1007/s12526-016-0606-y;,citation_issn=1867-1616, 1867-1624;,citation_volume=48;,citation_language=en-US;,citation_journal_title=Mar Biodiv;">
<meta name="citation_reference" content="citation_title=Accelerating ocean species discovery and laying the foundations for the future of marine biodiversity research and monitoring;,citation_abstract=Ocean Census is a new Large-Scale Strategic Science Mission aimed at accelerating the discovery and description of marine species. This mission addresses the knowledge gap of the diversity and distribution of marine life whereby of an estimated 1 million to 2 million species of marine life between 75% to 90% remain undescribed to date. Without improved knowledge of marine biodiversity, tackling the decline and eventual extinction of many marine species will not be possible. The marine biota has evolved over 4 billion years and includes many branches of the tree of life that do not exist on land or in freshwater. Understanding what is in the ocean and where it lives is fundamental science, which is required to understand how the ocean works, the direct and indirect benefits it provides to society and how human impacts can be reduced and managed to ensure marine ecosystems remain healthy. We describe a strategy to accelerate the rate of ocean species discovery by: 1) employing consistent standards for digitisation of species data to broaden access to biodiversity knowledge and enabling cybertaxonomy; 2) establishing new working practices and adopting advanced technologies to accelerate taxonomy; 3) building the capacity of stakeholders to undertake taxonomic and biodiversity research and capacity development, especially targeted at low- and middle-income countries (LMICs) so they can better assess and manage life in their waters and contribute to global biodiversity knowledge; and 4) increasing observational coverage on dedicated expeditions. Ocean Census, is conceived as a global open network of scientists anchored by Biodiversity Centres in developed countries and LMICs. Through a collaborative approach, including co-production of science with LMICs, and by working with funding partners, Ocean Census will focus and grow current efforts to discover ocean life globally, and permanently transform our ability to document, describe and safeguard marine species.;,citation_author=Alex David Rogers;,citation_author=Hannah Appiah-Madson;,citation_author=Jeff A. Ardron;,citation_author=Nicholas J. Bax;,citation_author=Punyasloke Bhadury;,citation_author=Angelika Brandt;,citation_author=Pier-Luigi Buttigieg;,citation_author=Olivier De Clerck;,citation_author=Claudia Delgado;,citation_author=Daniel L. Distel;,citation_author=Adrian Glover;,citation_author=Judith Gobin;,citation_author=Maila Guilhon;,citation_author=Shannon Hampton;,citation_author=Harriet Harden-Davies;,citation_author=Paul Hebert;,citation_author=Lisa Hynes;,citation_author=Miranda Lowe;,citation_author=Sandy MacIntyre;,citation_author=Hawis Madduppa;,citation_author=Ana Carolina De Azevedo Mazzuco;,citation_author=Anna McCallum;,citation_author=Chris McOwen;,citation_author=Tim Wilhelm Nattkemper;,citation_author=Mika Odido;,citation_author=Tim O’Hara;,citation_author=Karen Osborn;,citation_author=Angelique Pouponneau;,citation_author=Pieter Provoost;,citation_author=Muriel Rabone;,citation_author=Eva Ramirez-Llodra;,citation_author=Lucy Scott;,citation_author=Kerry Jennifer Sink;,citation_author=Daniela Turk;,citation_author=Hiromi Kayama Watanabe;,citation_author=Lauren V. Weatherdon;,citation_author=Thomas Wernberg;,citation_author=Suzanne Williams;,citation_author=Lucy Woodall;,citation_author=Dawn J. Wright;,citation_author=Daniela Zeppilli;,citation_author=Oliver Steeds;,citation_publication_date=2023-09;,citation_cover_date=2023-09;,citation_year=2023;,citation_fulltext_html_url=https://www.frontiersin.org/articles/10.3389/fmars.2023.1224471/full;,citation_doi=10.3389/fmars.2023.1224471;,citation_issn=2296-7745;,citation_volume=10;,citation_language=en-US;,citation_journal_title=Front. Mar. Sci.;">
<meta name="citation_reference" content="citation_title=Evidence for feminized genetic males in a flea beetle using newly identified X-linked markers;,citation_abstract=Abstract The equilibrium of sex ratios in sexually reproducing species is often disrupted by various environmental and genetic factors, including endosymbionts like Wolbachia . In this study, we explore the highly female-biased sex ratio observed in the flea beetle, Altica lythri , and its underlying mechanisms. Ancient hybridization events between Altica species have led to mitochondrial DNA introgression, resulting in distinct mitochondrial haplotypes that go along with different Wolbachia infections (HT1-wLytA1, HT1*- uninfected, HT2-wLytA2, and HT3-wLytB). Notably, beetles with some haplotypes exclusively produce female offspring, suggesting potential Wolbachia -induced phenomena such as feminization of genetic males. However, the observed female bias could also be a consequence of the ancient hybridization resulting in nuclear–cytoplasmic conflicts between introgressed mtDNA and nuclear genes. Through transcriptomic analysis and the program SEX-DETector, we established markers for genotypic sex differentiation for A. lythri , enabling genetic sexing via qPCR. Our findings suggest that feminization of genetic males is contributing to the skewed sex ratios, highlighting the intricate dynamics of sex determination and reproductive strategies in this flea beetle. This study provides valuable insights into the dynamics of genetic conflicts, endosymbionts, and sex ratios, revealing the novel phenomenon of genetic male feminization in the flea beetle A. lythri .;,citation_author=Kim Rohlfing;,citation_author=Malte Grewoldt;,citation_author=Mathilde Cordellier;,citation_author=Susanne Dobler;,citation_publication_date=2024-08;,citation_cover_date=2024-08;,citation_year=2024;,citation_fulltext_html_url=https://onlinelibrary.wiley.com/doi/10.1002/ece3.70123;,citation_issue=8;,citation_doi=10.1002/ece3.70123;,citation_issn=2045-7758, 2045-7758;,citation_volume=14;,citation_language=en-US;,citation_journal_title=Ecology and Evolution;">
<meta name="citation_reference" content="citation_title=TOUCH-UP Gradient Amplification Method;,citation_author=F. B. Rowther;,citation_author=H. Kardooni;,citation_author=T. Warr;,citation_publication_date=2012-04;,citation_cover_date=2012-04;,citation_year=2012;,citation_fulltext_html_url=http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3313699/;,citation_issue=1;,citation_doi=10.7171/jbt.12-2301-004;,citation_issn=1524-0215, 1943-4731;,citation_volume=23;,citation_journal_title=J Biomol Tech;">
<meta name="citation_reference" content="citation_title=On abyssal isopods (Crustaceae: Isopoda: Asellota) from the Angola Basin: Eurycope tumidicarpus n. Sp. And redescription of Acanthocope galatheae Wolff, 1962;,citation_author=Cornelia Schmid;,citation_author=Nils Brenke;,citation_author=Johann Wolfgang Wägele;,citation_publication_date=2002;,citation_cover_date=2002;,citation_year=2002;,citation_issue=1;,citation_volume=2;,citation_journal_title=Organisms Diversity &amp;amp;amp; Evolution;">
<meta name="citation_reference" content="citation_title=A new probabilistic method for quantifying &amp;amp;amp;lt;i&amp;gt;n&amp;lt;/i&amp;gt; -dimensional ecological niches and niche overlap;,citation_abstract=Considerable progress has been made in the development of statistical tools to quantify trophic relationships using stable isotope ratios, including tools that address size and overlap of isotopic niches. We build upon recent progress and propose a new probabilistic method for determining niche region and pairwise niche overlap that can be extended beyond two dimensions, provides directional estimates of niche overlap, accounts for species-specific distributions in niche space, and, unlike geometric methods, produces consistent and unique bivariate projections of multivariate data. We define the niche region (N R ) as a given 95% (or user-defined $\alpha$) probability region in multivariate space. Overlap is calculated as the probability that an individual from species A is found in the N R of species B . Uncertainty is accounted for in a Bayesian framework, and is the only aspect of the methodology that depends on sample size. Application is illustrated with three-dimensional stable isotope data, but practitioners could use any continuous indicator of ecological niche in any number of dimensions. We suggest that this represents an advance in our ability to quantify and compare ecological niches in a way that is more consistent with Hutchinson’s concept of an “ n -dimensional hypervolume.”;,citation_author=Heidi K. Swanson;,citation_author=Martin Lysy;,citation_author=Michael Power;,citation_author=Ashley D. Stasko;,citation_author=Jim D. Johnson;,citation_author=James D. Reist;,citation_publication_date=2015-02;,citation_cover_date=2015-02;,citation_year=2015;,citation_fulltext_html_url=https://esajournals.onlinelibrary.wiley.com/doi/10.1890/14-0235.1;,citation_issue=2;,citation_doi=10.1890/14-0235.1;,citation_issn=0012-9658, 1939-9170;,citation_volume=96;,citation_language=en-US;,citation_journal_title=Ecology;">
<meta name="citation_reference" content="citation_title=On the Metamorphoses of the Crustacea, and on Zoea, exposing their singular structure and demonstrating that they are not, as has been supposed, a peculiar Genus, but the Larva of Crustacea!! (1828);,citation_author=J. V. Thompson;,citation_publication_date=1968;,citation_cover_date=1968;,citation_year=1968;,citation_volume=2;,citation_inbook_title=Zoological Researches, and illustrations; or, natural history of nondescript or imperfectly known animals, in a series of memoirs, illustrated by numerous figures;,citation_series_title=Thompson, J.V. (1828-1834);">
<meta name="citation_reference" content="citation_title=Crustaceamorpha: Crustacean appendages;,citation_author=UC Museum of Paleontology;,citation_publication_date=2002-05;,citation_cover_date=2002-05;,citation_year=2002;,citation_fulltext_html_url=https://ucmp.berkeley.edu/arthropoda/crustacea/appendages.html;,citation_journal_title=Crustaceamorpha: Appendages;">
<meta name="citation_reference" content="citation_title=Crustaceamorpha: Metamorphosis and Larvae;,citation_author=UC Museum of Paleontology;,citation_publication_date=2002-05;,citation_cover_date=2002-05;,citation_year=2002;,citation_fulltext_html_url=https://ucmp.berkeley.edu/arthropoda/crustacea/metamorphosis.html#:~:text=Part%20of%20the%20larval%20period,minutes%20to%20over%20a%20year.;,citation_journal_title=Crustaceamorpha: Appendages;">
<meta name="citation_reference" content="citation_title=Bayesian statistics and modelling;,citation_author=Rens Van De Schoot;,citation_author=Sarah Depaoli;,citation_author=Ruth King;,citation_author=Bianca Kramer;,citation_author=Kaspar Märtens;,citation_author=Mahlet G. Tadesse;,citation_author=Marina Vannucci;,citation_author=Andrew Gelman;,citation_author=Duco Veen;,citation_author=Joukje Willemsen;,citation_author=Christopher Yau;,citation_publication_date=2021-01;,citation_cover_date=2021-01;,citation_year=2021;,citation_fulltext_html_url=https://www.nature.com/articles/s43586-020-00001-2;,citation_issue=1;,citation_doi=10.1038/s43586-020-00001-2;,citation_issn=2662-8449;,citation_volume=1;,citation_language=en-US;,citation_journal_title=Nat Rev Methods Primers;">
<meta name="citation_reference" content="citation_title=Hapsolutely: A user-friendly tool integrating haplotype phasing, network construction, and haploweb calculation;,citation_abstract=Abstract Motivation Haplotype networks are a routine approach to visualize relationships among alleles. Such visual analysis of single-locus data is still of importance, especially in species diagnosis and delimitation, where a limited amount of sequence data usually are available and sufficient, along with other datasets in the framework of integrative taxonomy. In diploid organisms, this often requires separating (phasing) sequences with heterozygotic positions, and typically separate programs are required for phasing, reformatting of input files, and haplotype network construction. We therefore developed Hapsolutely, a user-friendly program with an ergonomic graphical user interface that integrates haplotype phasing from single-locus sequences with five approaches for network/genealogy reconstruction. Results Among the novel options implemented, Hapsolutely integrates phasing and graphical reconstruction steps of haplotype networks, supports input of species partition data in the common SPART and SPART-XML formats, and calculates and visualizes haplowebs and fields for recombination, thus allowing graphical comparison of allele distribution and allele sharing among subsets for the purpose of species delimitation. The new tool has been specifically developed with a focus on the workflow in alpha-taxonomy, where exploring fields for recombination across alternative species partitions may help species delimitation. Availability and implementation Hapsolutely is written in Python, and integrates code from Phase, SeqPHASE, and PopART in C++ and Haxe. Compiled stand-alone executables for MS Windows and Mac OS along with a detailed manual can be downloaded from https://www.itaxotools.org; the source code is openly available on GitHub (https://github.com/iTaxoTools/Hapsolutely).;,citation_author=Miguel Vences;,citation_author=Stefanos Patmanidis;,citation_author=Jan-Christopher Schmidt;,citation_author=Michael Matschiner;,citation_author=Aurélien Miralles;,citation_author=Susanne S Renner;,citation_editor=Nicola Mulder;,citation_publication_date=2024-01;,citation_cover_date=2024-01;,citation_year=2024;,citation_fulltext_html_url=https://academic.oup.com/bioinformaticsadvances/article/doi/10.1093/bioadv/vbae083/7688355;,citation_issue=1;,citation_doi=10.1093/bioadv/vbae083;,citation_issn=2635-0041;,citation_volume=4;,citation_language=en-US;,citation_journal_title=Bioinformatics Advances;">
<meta name="citation_reference" content="citation_title=Searching factors causing implausible non-monophyly: Ssu rDNA phylogeny of Isopoda Asellota (Crustacea: Peracarida) and faster evolution in marine than in freshwater habitats;,citation_abstract=This contribution addresses two questions: which alignment patterns are causing non-monophyly of the Asellota and what is the phylogenetic history of this group? The Asellota are small benthic crustaceans occurring in most aquatic habitats. In viewof the complex morphological apomorphies known for this group, monophyly of the Asellota has never been questioned. Using ssu rDNA sequences of outgroups and of 16 asellote species from fresh water, littoral marine habitats and from deep-sea localities, the early divergence between the lineages in fresh water and in the ocean, and the monophyly of the deep-sea taxon Munnopsidae are confirmed. Relative substitution rates of freshwater species are much lower than in other isopod species, rates being highest in some littoral marine genera (Carpias and Jaera). Furthermore, more sequence sites are variable in marine than in freshwater species, the latter conserve outgroup character states. Monophyly is recovered with parsimony methods, but not with distance and maximum likelihood analyses, which tear apart the marine from the freshwater species. The information content of alignments was studied with spectra of supporting positions. The scarcity of signal ( $\frac{1}{4}$ apomorphic nucleotides) supporting monophyly of the Asellota is attributed to a short stem-line of this group or to erosion of signal in fast evolving marine species. Parametric boostrapping in combination with spectra indicates that a tree model cannot explain the data and that monophyly of the Asellota should not be rejected even though many topologies do not recover this taxon.;,citation_author=Johann-Wolfgang Wägele;,citation_author=Barbara Holland;,citation_author=Hermann Dreyer;,citation_author=Beate Hackethal;,citation_publication_date=2003-09;,citation_cover_date=2003-09;,citation_year=2003;,citation_fulltext_html_url=https://linkinghub.elsevier.com/retrieve/pii/S1055790303000538;,citation_issue=3;,citation_doi=10.1016/S1055-7903(03)00053-8;,citation_issn=10557903;,citation_volume=28;,citation_language=en-US;,citation_journal_title=Molecular Phylogenetics and Evolution;">
<meta name="citation_reference" content="citation_title=The systematics and biology of bathyal and abyssal Isopoda Asellota.;,citation_author=undefined Wolff;,citation_publication_date=1962;,citation_cover_date=1962;,citation_year=1962;,citation_volume=6;,citation_series_title=Galatheae Report;">
<meta name="citation_reference" content="citation_title=World register of marine species (WoRMS);,citation_abstract=The aim of the World Register of Marine Species (WoRMS) is to provide an authoritative and comprehensive list of names of marine organisms, including information on synonymy;,citation_author=S. Ahyong;,citation_author=C. B. Boyko;,citation_author=J. Bernot;,citation_author=S. N. Brandão;,citation_author=M. Daly;,citation_author=S. De Grave;,citation_author=N. J. Voogd;,citation_author=S. Gofas;,citation_author=F. Hernandez;,citation_author=L. Hughes;,citation_author=T. A. Neubauer;,citation_author=G. Paulay;,citation_author=S. Meij;,citation_author=B. Boydens;,citation_author=W. Decock;,citation_author=S. Dekeyzer;,citation_author=M. Goharimanesh;,citation_author=L. Vandepitte;,citation_author=B. Vanhoorne;,citation_author=R. Adlard;,citation_author=S. Agatha;,citation_author=K. J. Ahn;,citation_author=M. V. Alonso;,citation_author=B. Alvarez;,citation_author=M. R. W. Amler;,citation_author=V. Amorim;,citation_author=A. Anderberg;,citation_author=S. Andrés-Sánchez;,citation_author=Y. Ang;,citation_author=D. Antić;,citation_author=L. S.. Antonietto;,citation_author=C. Arango;,citation_author=T. Artois;,citation_author=S. Atkinson;,citation_author=K. Auffenberg;,citation_author=N. Bailly;,citation_author=B. G. Baldwin;,citation_author=R. Bank;,citation_author=A. Barber;,citation_author=R. L. Barrett;,citation_author=I. Bartsch;,citation_author=D. Bellan-Santini;,citation_author=N. Bergh;,citation_author=C. Bernard;,citation_author=F. J. Berrios Ortega;,citation_author=A. Berta;,citation_author=T. N. Bezerra;,citation_author=R. Bieler;,citation_author=S. Blanco;,citation_author=I. Blasco-Costa;,citation_author=M. Blazewicz;,citation_author=L. A. Błędzki;,citation_author=P. Bock;,citation_author=M. Bonifacino;,citation_author=R. Böttger-Schnack;,citation_author=P. Bouchet;,citation_author=N. Boury-Esnault;,citation_author=R. Bouzan;,citation_author=G. Boxshall;,citation_author=R. Bray;,citation_author=A. L. Brito Seixas;,citation_author=N. L. Bruce;,citation_author=A. Bruneau;,citation_author=N. Budaeva;,citation_author=J. Bueno-Villegas;,citation_author=J. Calvo Casas;,citation_author=P. Cárdenas;,citation_author=E. Carstens;,citation_author=P. Cartwright;,citation_author=T. Cedhagen;,citation_author=B. K. Chan;,citation_author=T. Y. Chan;,citation_author=A. Chernyshev;,citation_author=H. Choong;,citation_author=M. Christenhusz;,citation_author=M. Churchill;,citation_author=E. Cole;,citation_author=A. G. Collins;,citation_author=G. E. Collins;,citation_author=K. Collins;,citation_author=L. Consorti;,citation_author=D. Copilas-Ciocianu;,citation_author=L. Corbari;,citation_author=R. Cordeiro;,citation_author=V. M.d.M. Costa;,citation_author=P. H. Costa Corgosinho;,citation_author=M. Coste;,citation_author=K. A. Crandall;,citation_author=F. Cremonte;,citation_author=T. Cribb;,citation_author=S. Cutmore;,citation_author=F. Dahdouh-Guebas;,citation_author=M. Daneliya;,citation_author=J. C. Dauvin;,citation_author=P. Davie;,citation_author=C. De Broyer;,citation_author=P. Lima Ferreira;,citation_author=V. Mazancourt;,citation_author=L. Moura Oliveira;,citation_author=P. Decker;,citation_author=D. Defaye;,citation_author=H. Dekker;,citation_author=R. DeSalle;,citation_author=I. Di Capua;,citation_author=S. Dippenaar;,citation_author=M. Dohrmann;,citation_author=J. Dolan;,citation_author=D. Domning;,citation_author=R. D’Onofrio;,citation_author=R. Downey;,citation_author=N. Dreyer;,citation_author=U. Eisendle;,citation_author=M. Eitel;,citation_author=M. Eleaume;,citation_author=H. Enghoff;,citation_author=J. Epler;,citation_author=P. Esquete Garrote;,citation_author=N. L. Evenhuis;,citation_author=C. Ewers-Saucedo;,citation_author=M. Faber;,citation_author=D. Figueroa;,citation_author=C. Fišer;,citation_author=E. Fordyce;,citation_author=W. Foster;,citation_author=C. Fransen;,citation_author=S. Freire;,citation_author=S. Fujimoto;,citation_author=H. Furuya;,citation_author=M. Galbany-Casals;,citation_author=A. Gale;,citation_author=H. Galea;,citation_author=T. Gao;,citation_author=P. García-Moro;,citation_author=R. Garic;,citation_author=S. Garnett;,citation_author=S. Gaviria-Melo;,citation_author=S. Gerken;,citation_author=D. Gibson;,citation_author=R. Gibson;,citation_author=J. Gil;,citation_author=A. Gittenberger;,citation_author=C. Glasby;,citation_author=H. Glenner;,citation_author=A. Glover;,citation_author=S. E. Gómez-Noguera;,citation_author=A. I. Gondim;,citation_author=B. C. Gonzalez;,citation_author=D. González-Solís;,citation_author=C. Goodwin;,citation_author=M. Gostel;,citation_author=M. Grabowski;,citation_author=C. Gravili;,citation_author=M. Grossi;,citation_author=J. M.. Guerra-García;,citation_author=J. M. Guerrero;,citation_author=R. Guidetti;,citation_author=M. D. Guiry;,citation_author=D. Gutierrez;,citation_author=K. A. Hadfield;,citation_author=E. Hajdu;,citation_author=K. Halanych;,citation_author=J. Hallermann;,citation_author=B. W. Hayward;,citation_author=T. A. Hegna;,citation_author=G. Heiden;,citation_author=E. Hendrycks;,citation_author=D. Hennen;,citation_author=D. Herbert;,citation_author=A. Herrera Bachiller;,citation_author=M. Hodda;,citation_author=J. Høeg;,citation_author=B. Hoeksema;,citation_author=O. Holovachov;,citation_author=M. D. Hooge;,citation_author=J. N. Hooper;,citation_author=T. Horton;,citation_author=R. Houart;,citation_author=R. Huys;,citation_author=M. Hyžný;,citation_author=L. F. M. Iniesta;,citation_author=T. Iseto;,citation_author=M. Iwataki;,citation_author=R. Janssen;,citation_author=D. Jaume;,citation_author=K. Jazdzewski;,citation_author=C. D. Jersabek;,citation_author=P. Jiménez-Mejías;,citation_author=P. Jóźwiak;,citation_author=M. Jung;,citation_author=A. Kabat;,citation_author=H. Kajihara;,citation_author=K. Kakui;,citation_author=Y. Kantor;,citation_author=I. Karanovic;,citation_author=B. Karapunar;,citation_author=B. Karthick;,citation_author=J. Kathirithamby;,citation_author=L. Katinas;,citation_author=N. Kilian;,citation_author=Y. H. Kim;,citation_author=R. King;,citation_author=P. M. Kirk;,citation_author=M. Klautau;,citation_author=J. P. Kociolek;,citation_author=F. Köhler;,citation_author=K. Konowalik;,citation_author=A. Kotov;,citation_author=Z. Kovács;,citation_author=A. Kremenetskaia;,citation_author=R. M. Kristensen;,citation_author=A. Kroh;,citation_author=M. Kulikovskiy;,citation_author=S. Kullander;,citation_author=E. Kupriyanova;,citation_author=A. Lamaro;,citation_author=G. Lambert;,citation_author=I. Larridon;,citation_author=D. Lazarus;,citation_author=F. Le Coze;,citation_author=M. Le Roux;,citation_author=S. LeCroy;,citation_author=D. Leduc;,citation_author=E. J. Lefkowitz;,citation_author=R. Lemaitre;,citation_author=I. H. Lichter-Marck;,citation_author=S. C. Lim;,citation_author=D. Lindsay;,citation_author=Y. Liu;,citation_author=B. Loeuille;,citation_author=R. Lois;,citation_author=A. N. Lörz;,citation_author=M. Luceño Garcés;,citation_author=T. Ludwig;,citation_author=N. Lundholm;,citation_author=E. Macpherson;,citation_author=E. Maguilla;,citation_author=C. Mah;,citation_author=T. Mamos;,citation_author=R. Manconi;,citation_author=G. Mapstone;,citation_author=P. E. Marek;,citation_author=K. Markello;,citation_author=J. I. Márquez-Corro;,citation_author=B. Marshall;,citation_author=D. J. Marshall;,citation_author=P. Martin;,citation_author=S. Martín-Bravo;,citation_author=P. Martinez Arbizu;,citation_author=S. Maslakova;,citation_author=C. McFadden;,citation_author=S. J. McInnes;,citation_author=R. McKenzie;,citation_author=J. Means;,citation_author=J. Mees;,citation_author=H. H. Mejía-Madrid;,citation_author=K. Meland;,citation_author=K. L. Merrin;,citation_author=M. Míguez;,citation_author=J. Miller;,citation_author=C. Mills;,citation_author=Ø. Moestrup;,citation_author=V. Mokievsky;,citation_author=T. Molodtsova;,citation_author=R. Mooi;,citation_author=A. Morales-Alonso;,citation_author=A. C. Morandini;,citation_author=R. Moreira da Rocha;,citation_author=L. D. Moreyra C.;,citation_author=L. Moritz;,citation_author=C. Morrow;,citation_author=J. Mortelmans;,citation_author=A. Müller;,citation_author=A. R. Muñoz Gallego;,citation_author=P. Muñoz Schüler;,citation_author=L. Musco;,citation_author=J. B. Nascimento;,citation_author=G. Nesom;,citation_author=M.d.S. Neto Silva;,citation_author=E. Neubert;,citation_author=B. Neuhaus;,citation_author=P. Ng;,citation_author=A. D. Nguyen;,citation_author=C. Nielsen;,citation_author=S. Nielsen;,citation_author=T. Nishikawa;,citation_author=J. Norenburg;,citation_author=T. O’Hara;,citation_author=D. Opresko;,citation_author=M. Osawa;,citation_author=H. J. Osigus;,citation_author=Y. Ota;,citation_author=B. Páll-Gergely;,citation_author=J. L. Panero;,citation_author=A. Parra-Gómez;,citation_author=D. Patterson;,citation_author=M. Pedram;,citation_author=P. Pelser;,citation_author=R. Peña Santiago;,citation_author=G. Perbiche-Neves;,citation_author=J.d.S.. Pereira;,citation_author=P. H. M. Pereira;,citation_author=S. G. G. Pereira;,citation_author=M. Perez-Losada;,citation_author=I. Petrescu;,citation_author=T. Pfingstl;,citation_author=W. Piasecki;,citation_author=D. Pica;,citation_author=B. Picton;,citation_author=J. Pignatti;,citation_author=J. F. Pilger;,citation_author=U. Pinheiro;,citation_author=A. B. Pisera;,citation_author=B. Poatskievick Pierezan;,citation_author=D. Polhemus;,citation_author=G. C. Poore;,citation_author=M. Potapova;,citation_author=R. A. Praxedes;,citation_author=V. Půža;,citation_author=G. Read;,citation_author=M. Reich;,citation_author=J. D. Reimer;,citation_author=H. Reip;,citation_author=V. Resende Bueno;,citation_author=M. Reuscher;,citation_author=J. W. Reynolds;,citation_author=I. Richling;,citation_author=F. Rimet;,citation_author=P. Ríos;,citation_author=M. Rius;,citation_author=E. Rodríguez;,citation_author=D. C. Rogers;,citation_author=N. Roque;,citation_author=G. Rosenberg;,citation_author=K. Rützler;,citation_author=I. R.d.S. Sa;,citation_author=H. A. B.. Sá;,citation_author=M. Saavedra;,citation_author=L. M. Sabater;,citation_author=K. Sabbe;,citation_author=R. Sabroux;,citation_author=J. Saiz-Salinas;,citation_author=S. Sala;,citation_author=K. Samimi-Namin;,citation_author=R. Sánchez-Villegas;,citation_author=K. Sangtae;,citation_author=S. Santagata;,citation_author=S. Santos;,citation_author=S. G. Santos;,citation_author=M. A. B.d. Santos Filho;,citation_author=M. Sanz Arnal;,citation_author=E. Sar;,citation_author=T. Saucède;,citation_author=L. Schärer;,citation_author=B. Schierwater;,citation_author=E. Schilling;,citation_author=A. Schmidt-Lebuhn;,citation_author=S. Schneider;,citation_author=C. Schönberg;,citation_author=J. Schrével;,citation_author=P. Schuchert;,citation_author=C. Schweitzer;,citation_author=J. C. Semple;,citation_author=A. R. Senna;,citation_author=A. Sennikov;,citation_author=C. Serejo;,citation_author=S. Shaik;,citation_author=S. Shamsi;,citation_author=J. Sharma;,citation_author=W. A. Shear;,citation_author=N. Shenkar;,citation_author=M. Short;,citation_author=J. Sicinski;,citation_author=P. Sierwald;,citation_author=D. K. F.d. Silva;,citation_author=E. S. S. Silva;,citation_author=M. L. C. N. Silva;,citation_author=E. Simmons;,citation_author=F. Sinniger;,citation_author=C. Sinou;,citation_author=D. Sivell;,citation_author=H. Smit;,citation_author=N. Smit;,citation_author=N. Smol;,citation_author=M. V. Sørensen;,citation_author=J. F.. Souza-Filho;,citation_author=J. Spelda;,citation_author=W. Sterrer;,citation_author=H. M. Steyn;,citation_author=P. Stoev;,citation_author=S. Stöhr;,citation_author=E. Suárez-Morales;,citation_author=A. Susanna;,citation_author=C. Suttle;,citation_author=B. J. Swalla;,citation_author=M. Tanaka;,citation_author=A. H. Tandberg;,citation_author=D. Tang;,citation_author=M. Tasker;,citation_author=J. Taylor;,citation_author=J. Taylor;,citation_author=K. Taylor;,citation_author=A. Tchesunov;,citation_author=E. Temereva;,citation_author=H. Hove;,citation_author=J. J. Poorten;,citation_author=K. Thirouin;,citation_author=J. D. Thomas;,citation_author=E. V. Thuesen;,citation_author=M. Thurston;,citation_author=B. Thuy;,citation_author=J. T. Timi;,citation_author=A. Todaro;,citation_author=J. Todd;,citation_author=X. Turon;,citation_author=S. Tyler;,citation_author=P. Uetz;,citation_author=L. Urbatsch;,citation_author=J. Uribe-Palomino;,citation_author=E. Urtubey;,citation_author=S. Utevsky;,citation_author=J. Vacelet;,citation_author=W. Vader;,citation_author=R. Väinölä;,citation_author=A. Valdés-Florido;,citation_author=G. Valls Domedel;,citation_author=B. Van de Vijver;,citation_author=T. Haaren;,citation_author=R. W. Soest;,citation_author=A. Vanreusel;,citation_author=B. Vázquez-García;,citation_author=J. Velandia;,citation_author=V. Venekey;,citation_author=T. Verhoeff;,citation_author=T. Villaverde;,citation_author=M. Vinarski;,citation_author=R. Vonk;,citation_author=C. Vos;,citation_author=A. A. Vouilloud;,citation_author=G. Walker-Smith;,citation_author=T. C. Walter;,citation_author=L. Watling;,citation_author=M. Wayland;,citation_author=T. Wesener;,citation_author=C. E. Wetzel;,citation_author=C. Whipps;,citation_author=K. White;,citation_author=U. Wieneke;,citation_author=D. M. Williams;,citation_author=G. Williams;,citation_author=N. Williams;,citation_author=K. L. Wilson;,citation_author=R. Wilson;,citation_author=J. Witkowski;,citation_author=J. Xavier;,citation_author=K. Xu;,citation_author=J. Zanol;,citation_author=W. Zeidler;,citation_author=Z. Zhao;,citation_author=A. Zullini;,citation_fulltext_html_url=https://www.marinespecies.org;,citation_publisher=$&amp;amp;amp;lt;$a href=&amp;quot;=&amp;quot;$&amp;gt;$=$&amp;lt;$/a$&amp;gt;$https://www.marinespecies.org; WoRMS Editorial Board;">
<meta name="citation_reference" content="citation_title=Benthic carbon fixation and cycling in diffuse hydrothermal and background sediments in the Bransfield Strait, Antarctica;,citation_abstract=Abstract. Sedimented hydrothermal vents are likely to be widespread compared to hard substrate hot vents. They host chemosynthetic microbial communities which fix inorganic carbon (C) at the seafloor, as well as a wide range of macroinfauna, including vent-obligate and background non-vent taxa. There are no previous direct observations of carbon cycling at a sedimented hydrothermal vent. We conducted 13C isotope tracing experiments at three sedimented sites in the Bransfield Strait, Antarctica, which showed different degrees of hydrothermalism. Two experimental treatments were applied, with 13C added as either algal detritus (photosynthetic C), or as bicarbonate (substrate for benthic C fixation). Algal 13C was taken up by both bacteria and metazoan macrofaunal, but its dominant fate was respiration, as observed at deeper and more food-limited sites elsewhere. Rates of 13C uptake and respiration suggested that the diffuse hydrothermal site was not the hot spot of benthic C cycling that we hypothesised it would be. Fixation of inorganic C into bacterial biomass was observed at all sites, and was measurable at two out of three sites. At all sites, newly fixed C was transferred to metazoan macrofauna. Fixation rates were relatively low compared with similar experiments elsewhere; thus, C fixed at the seafloor was a minor C source for the benthic ecosystem. However, as the greatest amount of benthic C fixation occurred at the “Off Vent” (non-hydrothermal) site (0.0770.034 mg C m-2 fixed during 60 h), we suggest that benthic fixation of inorganic C is more widespread than previously thought, and warrants further study.;,citation_author=Clare Woulds;,citation_author=James B. Bell;,citation_author=Adrian G. Glover;,citation_author=Steven Bouillon;,citation_author=Louise S. Brown;,citation_publication_date=2020-01;,citation_cover_date=2020-01;,citation_year=2020;,citation_fulltext_html_url=https://bg.copernicus.org/articles/17/1/2020/;,citation_issue=1;,citation_doi=10.5194/bg-17-1-2020;,citation_issn=1726-4189;,citation_volume=17;,citation_language=en-US;,citation_journal_title=Biogeosciences;">
<meta name="citation_reference" content="citation_title=Environmental niche models improve species identification in &amp;amp;amp;lt;span style=&amp;quot;font-variant:small-caps;&amp;quot;&amp;gt;DNA&amp;lt;/span&amp;gt; barcoding;,citation_abstract=Abstract Recent advances in DNA barcoding have immeasurably advanced global biodiversity research in the last two decades. However, inherent limitations in barcode sequences, such as hybridization, introgression or incomplete lineage sorting can lead to misidentifications when relying solely on barcode sequences. Here, we propose a new Niche-model-Based Species Identification (NBSI) method based on the idea that species distribution information is a potential complement to DNA barcoding species identifications. NBSI performs species membership inference by incorporating niche modelling predictions and traditional DNA barcoding identifications. Systematic tests across diverse scenarios show significant improvements in species identification success rates under the newly proposed NBSI framework, where the largest increase is from 4.7% (95% CI: 3.51%–6.25%) to 94.8% (95% CI: 93.19%–96.06%). Additionally, obvious improvements were observed when using NBSI on potentially ambiguous sequences whose genetic nearest neighbours belongs to another species or more than two species, which occurs commonly with species represented by single or short DNA barcodes. These results support our assertion that environmental factors/variables are valuable complements to DNA sequence data for species identification by avoiding potential misidentifications inferred from genetic information alone. The NBSI framework is currently implemented as a new R package, “NicheBarcoding”, that is open source under GNU General Public Licence and freely available from https://CRAN.R-project.org/package=NicheBarcoding . , 摘要 近二十年来，DNA条形码技术的发展极大推进了全球生物多样性的研究进程。然而，条形码序列固有的局限性，如杂交、基因渐渗或不完全谱系分选，可能导致仅依赖条形码序列进行物种鉴定时出现错误。 在本文中，我们提出了一种基于生态位模型的物种鉴定新方法（NBSI），该方法认为物种的分布信息是DNA条形码物种鉴定的潜在补充。NBSI将生态位模型的预测与传统DNA条形码鉴定相整合，从而推断未知样本所属的物种。 通过对多种情况的数据集进行系统性测试，结果显示在新提出的NBSI框架下，物种鉴定成功率显著提高，其中提升幅度最大的是从4.7%（95%CI：3.51%-6.25%）提高至 94.8%（95%CI：93.19%-96.06%）。此外，在使用NBSI处理潜在错误序列时也观察到了明显的改进，这些序列的遗传最近邻体属于除它本身之外的另一个物种或属于两个以上的物种，这类情况在单序列物种或短序列物种数据集中很常见。 上述结果支持了我们的观点，即环境因素/变量可以作为DNA序列数据的有益补充，通过避免仅依赖遗传信息得出的潜在错误判断来进行物种鉴定。NBSI框架目前已编写为一个新的R语言函数包“NicheBarcoding”实现，该包在GNU通用公共许可证下开源，并且可以从 https://CRAN.R-project.org/package=NicheBarcoding 免费获取。;,citation_author=Cai-qing Yang;,citation_author=Ying Wang;,citation_author=Xin-hai Li;,citation_author=Jing Li;,citation_author=Bing Yang;,citation_author=Michael C. Orr;,citation_author=Ai-bing Zhang;,citation_publication_date=2024-12;,citation_cover_date=2024-12;,citation_year=2024;,citation_fulltext_html_url=https://besjournals.onlinelibrary.wiley.com/doi/10.1111/2041-210X.14440;,citation_issue=12;,citation_doi=10.1111/2041-210X.14440;,citation_issn=2041-210X, 2041-210X;,citation_volume=15;,citation_language=en-US;,citation_journal_title=Methods Ecol Evol;">
<meta name="citation_reference" content="citation_title=geneHapR: An R package for gene haplotypic statistics and visualization;,citation_abstract=Background: Together with application of next-generation sequencing technologies and increased accumulation of genomic variation data in different organism species, an opportunity for effectively identification of superior alleles of functional genes to facilitate marker-assisted selection is emerging, and the clarification of haplotypes of functional genes is becoming an essential target in recent study works. Results: In this paper, we describe an R package “geneHapR” developed for haplotypes identification, statistics and visualization analysis of candidate genes. This package could integrate genotype data, genomic annotating information and phenotypic variation data to clarify genotype variations, evolutionary-ship, and morphological effects among haplotypes through variants visualization, network construction and phenotypic comparison. “geneHapR” also provides functions for Linkage Disequilibrium block analysis and visualizing of haplotypes geo-distribution. Conclusions: The R package “geneHapR” provided an easy-to-use tool for haplotype identification, statistic and visualization for candidate gene and will provide useful clues for gene functional dissection and molecular-assistant pyramiding of beneficial alleles of functional locus in future breeding programs.;,citation_author=Renliang Zhang;,citation_author=Guanqing Jia;,citation_author=Xianmin Diao;,citation_publication_date=2023-05;,citation_cover_date=2023-05;,citation_year=2023;,citation_fulltext_html_url=https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-023-05318-9;,citation_issue=1;,citation_doi=10.1186/s12859-023-05318-9;,citation_issn=1471-2105;,citation_volume=24;,citation_language=en-US;,citation_journal_title=BMC Bioinformatics;">
</head>

<body>

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <div class="quarto-title-block"><div><h1 class="title">Integrative Taxonomy of North Atlantic <em>Acanthocope</em> with description of a new species</h1><button type="button" class="btn code-tools-button" id="quarto-code-tools-source"><i class="bi"></i></button></div></div>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Authors</div>
          <div class="quarto-title-meta-heading">Affiliations</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Zhehao Hu <a href="mailto:hu_zhehao@hotmail.com" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> </p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        University of Hamburg
                      </p>
                  </div>
                      <div class="quarto-title-meta-contents">
            <p class="author">Jenny Neuhaus <a href="mailto:jenny.neuhaus@senckenberg.de" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0009-0000-7570-4094" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        Senckenberg am Meer
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      <div class="quarto-alternate-formats"><div class="quarto-title-meta-heading">Other Formats</div><div class="quarto-title-meta-contents"><p><a href="index.docx"><i class="bi bi-file-word"></i>MS Word (apaquarto)</a></p></div><div class="quarto-title-meta-contents"><p><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF (apaquarto)</a></p></div><div class="quarto-title-meta-contents"><p><a href="index-meca.zip" data-meca-link="true"><i class="bi bi-archive"></i>MECA Bundle</a></p></div></div></div>
    </div>


    <div>
      <div class="keywords">
        <div class="block-title">Keywords</div>
        <p>Integrative taxonomy, Isopoda, Deep-sea, Biogeography</p>
      </div>
    </div>

    <div class="quarto-other-links-text-target">
    </div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active" data-toc-expanded="2">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#about-the-manuscript" id="toc-about-the-manuscript" class="nav-link active" data-scroll-target="#about-the-manuscript">About the manuscript</a></li>
  <li><a href="#whiteboard" id="toc-whiteboard" class="nav-link" data-scroll-target="#whiteboard">Whiteboard</a>
  <ul>
  <li><a href="#s-and-tcs-abstract" id="toc-s-and-tcs-abstract" class="nav-link" data-scroll-target="#s-and-tcs-abstract">03-07 | 18S and TCS Abstract</a></li>
  <li><a href="#loan-request-zmh-pending" id="toc-loan-request-zmh-pending" class="nav-link" data-scroll-target="#loan-request-zmh-pending">02-28 | Loan request ZMH (Pending)</a></li>
  </ul></li>
  <li><a href="#abstract" id="toc-abstract" class="nav-link" data-scroll-target="#abstract">Abstract</a></li>
  </ul>
<div class="quarto-alternate-notebooks"><h2>Notebooks</h2><ul><li><a href="notebooks/10 Abstract-preview.html"><i class="bi bi-journal-code"></i>10 Manuscript Abstract</a></li><li><a href="notebooks/11 TCS-Abstract-preview.html"><i class="bi bi-journal-code"></i>11 TCS Abstract</a></li><li><a href="notebooks/20 Introduction-preview.html"><i class="bi bi-journal-code"></i>20 Introduction</a></li><li><a href="notebooks/42 Result - Genetics-preview.html"><i class="bi bi-journal-code"></i>42 Result - Genetics</a></li><li><a href="notebooks/51 Drawing-preview.html"><i class="bi bi-journal-code"></i>51 Drawing</a></li></ul></div></nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">



  


<section id="about-the-manuscript" class="level2">
<h2 class="anchored" data-anchor-id="about-the-manuscript">About the manuscript</h2>
<p>Latest edition is online at <a href="https://zzzhehao.github.io/ACTCP/#ACTCPFELIS">https://zzzhehao.github.io/ACTCP/</a></p>
<p>Password required: <code>ACTCPFELIS</code></p>
<p><strong>Download</strong></p>
<ul>
<li><strong>MS Word</strong> (APA 7 style manuscript), <strong>PDF</strong> (APA 7 style manuscript) and <strong>MECA</strong> bundle (Computation source code included) are available under “Other Formats” in the header banner</li>
</ul>
<p><strong>Commenting</strong></p>
<ul>
<li>Register <a href="https://hypothes.is/groups/6KpDgDmQ/actcp">here (Hypothes.is)</a></li>
<li>Use annotation to add comments, make sure to change group to ACTCP (private).</li>
<li>Or send comments, suggestion, edited MS Word file to <a href="mailto:hu_zhehao@hotmail.com">hu_zhehao@hotmail.com</a></li>
</ul>
</section>
<section id="whiteboard" class="level2">
<h2 class="anchored" data-anchor-id="whiteboard">Whiteboard</h2>
<section id="s-and-tcs-abstract" class="level3">
<h3 class="anchored" data-anchor-id="s-and-tcs-abstract">03-07 | 18S and TCS Abstract</h3>
<ul>
<li>A quick preview to the 18S phylogeny: <a href="./notebooks/42 Result - Genetics-preview.html">genetic result</a></li>
<li>Abstract for TCS Meeting Paris: <a href="./notebooks/11 TCS-Abstract-preview.html">TCS Abstract</a></li>
<li>All drawing scans are <a href="./notebooks/51 Drawing-preview.html">here</a></li>
</ul>
</section>
<section id="loan-request-zmh-pending" class="level3">
<h3 class="anchored" data-anchor-id="loan-request-zmh-pending">02-28 | Loan request ZMH (Pending)</h3>
<ol type="1">
<li><strong>Type materials</strong></li>
</ol>
<ul>
<li><em>A. puertoricana</em>: ZMH K–46195 (Holotype), ZMH K–46196, ZMH K–46198 (Puerto Rico Trench)</li>
<li><em>A. eleganta</em>: ZMH K–40716 (Holotype), ZMH K–40717, ZMH K–40718. (Weddell Sea)</li>
</ul>
<ol start="2" type="1">
<li><strong>ZMH K–46200</strong>: <em>A. eleganta</em></li>
</ol>
<p><em>A. eleganta</em> morphologically identified by <span class="citation" data-cites="malyutina2018">Malyutina, Frutos, and Brandt (<a href="#ref-malyutina2018" role="doc-biblioref">2018</a>)</span>, found at the same station as <em>A. puertoricana</em> (Puerto Rico Trench). A big support that <em>A. eleganta</em> and <em>A. puertoricana</em> are the same species if we can pull genetics and proteomics from them.</p>
<ol start="3" type="1">
<li><strong>ZMH K–46199</strong>: <em>A. annulatus</em></li>
</ol>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Note
</div>
</div>
<div class="callout-body-container callout-body">
<p><span class="citation" data-cites="malyutina2018">Malyutina, Frutos, and Brandt (<a href="#ref-malyutina2018" role="doc-biblioref">2018</a>)</span>:</p>
<p>Acanthocope annulatus Menzies, 1962 (Fig. 16)</p>
<p>Acanthocope annulatus (Menzies, 1962: 155, Fig. 44 B, C; Malyutina and Brandt, 2004: 11, Figs. 6–10. Material examined: 1 male (5.9 mm TL), ZMH K–46199, Puerto Rico Trench abyssal area, RV Sonne, Vema-TRANSIT cruise, 25 January 2015, C-EBS, station 14-1, 19°1.63′N 67°9.73′W, 4552‒4552 m depth, supranet, bottom temperature: 2.25 °C, bottom O2: 261.13 μM, bottom current: 5.25 cm/s.</p>
<p>Remarks: The only specimen of the species, found at the same station as A. puertoricana sp. nov., was identified as it has all described characters of the Acanthocope annulatus Menzies, 1962. This new record is located at more than 10000 km far away from the type locality in the southeastern Atlantic (SW of Cape Town). Future sampling in the Atlantic and genetic investigations might shed some light on the true distribution of this species.</p>
</div>
</div>
</section>
</section>
<section id="abstract" class="level2">
<h2 class="anchored" data-anchor-id="abstract">Abstract</h2>
<p>Science still struggles with reliable taxonomy that forms the foundation for many subsequent studies on marine ecosystems. A case study with integrative approach reports first phylogenetic understanding of the genus <em>Acanthocope</em> (Isopoda: Munnopsidae) as well as its biogeography in the North Atlantic, in addition with the description of a new species.</p>
<p>(… more with results: Hybridization of Morphogroup 2? Confirm of monophyly? etc.)</p>
<p>(… what impact does this have? “Contributing to current understanding of evolution pattern of Munnopsidae”??)</p>
<!-- -->


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography" id="quarto-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-malyutina2018" class="csl-entry" role="listitem">
Malyutina, Marina V., Inmaculada Frutos, and Angelika Brandt. 2018. <span>“Diversity and Distribution of the Deep-Sea <span>Atlantic Acanthocope</span> (<span>Crustacea</span>, <span>Isopoda</span>, <span>Munnopsidae</span>), with Description of Two New Species.”</span> <em>Deep Sea Research Part II: Topical Studies in Oceanography</em> 148 (February): 130–50. <a href="https://doi.org/10.1016/j.dsr2.2017.11.003">https://doi.org/10.1016/j.dsr2.2017.11.003</a>.
</div>
</div></section></div></main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const disableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'prefetch';
    }
  }
  const enableStylesheet = (stylesheets) => {
    for (let i=0; i < stylesheets.length; i++) {
      const stylesheet = stylesheets[i];
      stylesheet.rel = 'stylesheet';
    }
  }
  const manageTransitions = (selector, allowTransitions) => {
    const els = window.document.querySelectorAll(selector);
    for (let i=0; i < els.length; i++) {
      const el = els[i];
      if (allowTransitions) {
        el.classList.remove('notransition');
      } else {
        el.classList.add('notransition');
      }
    }
  }
  const toggleGiscusIfUsed = (isAlternate, darkModeDefault) => {
    const baseTheme = document.querySelector('#giscus-base-theme')?.value ?? 'light';
    const alternateTheme = document.querySelector('#giscus-alt-theme')?.value ?? 'dark';
    let newTheme = '';
    if(darkModeDefault) {
      newTheme = isAlternate ? baseTheme : alternateTheme;
    } else {
      newTheme = isAlternate ? alternateTheme : baseTheme;
    }
    const changeGiscusTheme = () => {
      // From: https://github.com/giscus/giscus/issues/336
      const sendMessage = (message) => {
        const iframe = document.querySelector('iframe.giscus-frame');
        if (!iframe) return;
        iframe.contentWindow.postMessage({ giscus: message }, 'https://giscus.app');
      }
      sendMessage({
        setConfig: {
          theme: newTheme
        }
      });
    }
    const isGiscussLoaded = window.document.querySelector('iframe.giscus-frame') !== null;
    if (isGiscussLoaded) {
      changeGiscusTheme();
    }
  }
  const toggleColorMode = (alternate) => {
    // Switch the stylesheets
    const alternateStylesheets = window.document.querySelectorAll('link.quarto-color-scheme.quarto-color-alternate');
    manageTransitions('#quarto-margin-sidebar .nav-link', false);
    if (alternate) {
      enableStylesheet(alternateStylesheets);
      for (const sheetNode of alternateStylesheets) {
        if (sheetNode.id === "quarto-bootstrap") {
          toggleBodyColorMode(sheetNode);
        }
      }
    } else {
      disableStylesheet(alternateStylesheets);
      toggleBodyColorPrimary();
    }
    manageTransitions('#quarto-margin-sidebar .nav-link', true);
    // Switch the toggles
    const toggles = window.document.querySelectorAll('.quarto-color-scheme-toggle');
    for (let i=0; i < toggles.length; i++) {
      const toggle = toggles[i];
      if (toggle) {
        if (alternate) {
          toggle.classList.add("alternate");     
        } else {
          toggle.classList.remove("alternate");
        }
      }
    }
    // Hack to workaround the fact that safari doesn't
    // properly recolor the scrollbar when toggling (#1455)
    if (navigator.userAgent.indexOf('Safari') > 0 && navigator.userAgent.indexOf('Chrome') == -1) {
      manageTransitions("body", false);
      window.scrollTo(0, 1);
      setTimeout(() => {
        window.scrollTo(0, 0);
        manageTransitions("body", true);
      }, 40);  
    }
  }
  const isFileUrl = () => { 
    return window.location.protocol === 'file:';
  }
  const hasAlternateSentinel = () => {  
    let styleSentinel = getColorSchemeSentinel();
    if (styleSentinel !== null) {
      return styleSentinel === "alternate";
    } else {
      return false;
    }
  }
  const setStyleSentinel = (alternate) => {
    const value = alternate ? "alternate" : "default";
    if (!isFileUrl()) {
      window.localStorage.setItem("quarto-color-scheme", value);
    } else {
      localAlternateSentinel = value;
    }
  }
  const getColorSchemeSentinel = () => {
    if (!isFileUrl()) {
      const storageValue = window.localStorage.getItem("quarto-color-scheme");
      return storageValue != null ? storageValue : localAlternateSentinel;
    } else {
      return localAlternateSentinel;
    }
  }
  const darkModeDefault = false;
  let localAlternateSentinel = darkModeDefault ? 'alternate' : 'default';
  // Dark / light mode switch
  window.quartoToggleColorScheme = () => {
    // Read the current dark / light value 
    let toAlternate = !hasAlternateSentinel();
    toggleColorMode(toAlternate);
    setStyleSentinel(toAlternate);
    toggleGiscusIfUsed(toAlternate, darkModeDefault);
  };
  // Ensure there is a toggle, if there isn't float one in the top right
  if (window.document.querySelector('.quarto-color-scheme-toggle') === null) {
    const a = window.document.createElement('a');
    a.classList.add('top-right');
    a.classList.add('quarto-color-scheme-toggle');
    a.href = "";
    a.onclick = function() { try { window.quartoToggleColorScheme(); } catch {} return false; };
    const i = window.document.createElement("i");
    i.classList.add('bi');
    a.appendChild(i);
    window.document.body.appendChild(a);
  }
  // Switch to dark mode if need be
  if (hasAlternateSentinel()) {
    toggleColorMode(true);
  } else {
    toggleColorMode(false);
  }
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb1" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Integrative Taxonomy of North Atlantic *Acanthocope* with description of a new species"</span></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a><span class="an">shorttitle:</span><span class="co"> "ACTCP"</span></span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a><span class="an">author:</span></span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="co">  - name: Zhehao Hu</span></span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a><span class="co">    corresponding: true</span></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="co">    # orcid: </span></span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a><span class="co">    email: "hu_zhehao@hotmail.com"</span></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a><span class="co">    affiliations:</span></span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a><span class="co">      - name: University of Hamburg</span></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="co">        department: Department of Biology</span></span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a><span class="co">        address: Martin-Luther-King Platz 3</span></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a><span class="co">        city: Hamburg</span></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a><span class="co">        postal-code: D-20146</span></span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a><span class="co">  - name: Jenny Neuhaus</span></span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a><span class="co">    orcid: 0009-0000-7570-4094</span></span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a><span class="co">    email: "jenny.neuhaus@senckenberg.de"</span></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="co">    affiliations:</span></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="co">      - name: Senckenberg am Meer</span></span>
<span id="cb1-22"><a href="#cb1-22" aria-hidden="true" tabindex="-1"></a><span class="co">        department: German Centre for Marine Biodiversity Research</span></span>
<span id="cb1-23"><a href="#cb1-23" aria-hidden="true" tabindex="-1"></a><span class="co">        address: Martin-Luther-King Platz 3</span></span>
<span id="cb1-24"><a href="#cb1-24" aria-hidden="true" tabindex="-1"></a><span class="co">        city: Hamburg</span></span>
<span id="cb1-25"><a href="#cb1-25" aria-hidden="true" tabindex="-1"></a><span class="co">        postal-code: D-20146</span></span>
<span id="cb1-26"><a href="#cb1-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-27"><a href="#cb1-27" aria-hidden="true" tabindex="-1"></a><span class="co"># abstract: "Acanthocope"</span></span>
<span id="cb1-28"><a href="#cb1-28" aria-hidden="true" tabindex="-1"></a><span class="an">keywords:</span><span class="co"> [Integrative taxonomy, Isopoda, Deep-sea, Biogeography]</span></span>
<span id="cb1-29"><a href="#cb1-29" aria-hidden="true" tabindex="-1"></a><span class="an">author-note:</span></span>
<span id="cb1-30"><a href="#cb1-30" aria-hidden="true" tabindex="-1"></a><span class="co">  disclosures:</span></span>
<span id="cb1-31"><a href="#cb1-31" aria-hidden="true" tabindex="-1"></a><span class="co">    conflict of interest: The author has no conflict of interest to declare.</span></span>
<span id="cb1-32"><a href="#cb1-32" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb1-33"><a href="#cb1-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-34"><a href="#cb1-34" aria-hidden="true" tabindex="-1"></a><span class="fu">## About the manuscript</span></span>
<span id="cb1-35"><a href="#cb1-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-36"><a href="#cb1-36" aria-hidden="true" tabindex="-1"></a>Latest edition is online at <span class="co">[</span><span class="ot">https://zzzhehao.github.io/ACTCP/</span><span class="co">](https://zzzhehao.github.io/ACTCP/#ACTCPFELIS)</span></span>
<span id="cb1-37"><a href="#cb1-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-38"><a href="#cb1-38" aria-hidden="true" tabindex="-1"></a>Password required: <span class="in">`ACTCPFELIS`</span></span>
<span id="cb1-39"><a href="#cb1-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-40"><a href="#cb1-40" aria-hidden="true" tabindex="-1"></a>**Download**</span>
<span id="cb1-41"><a href="#cb1-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-42"><a href="#cb1-42" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**MS Word** (APA 7 style manuscript), **PDF** (APA 7 style manuscript) and **MECA** bundle (Computation source code included) are available under "Other Formats" in the header banner</span>
<span id="cb1-43"><a href="#cb1-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-44"><a href="#cb1-44" aria-hidden="true" tabindex="-1"></a>**Commenting**</span>
<span id="cb1-45"><a href="#cb1-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-46"><a href="#cb1-46" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Register <span class="co">[</span><span class="ot">here (Hypothes.is)</span><span class="co">](https://hypothes.is/groups/6KpDgDmQ/actcp)</span></span>
<span id="cb1-47"><a href="#cb1-47" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Use annotation to add comments, make sure to change group to ACTCP (private).</span>
<span id="cb1-48"><a href="#cb1-48" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Or send comments, suggestion, edited MS Word file to <span class="co">[</span><span class="ot">hu_zhehao@hotmail.com</span><span class="co">](mailto:hu_zhehao@hotmail.com)</span></span>
<span id="cb1-49"><a href="#cb1-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-50"><a href="#cb1-50" aria-hidden="true" tabindex="-1"></a><span class="fu">## Whiteboard</span></span>
<span id="cb1-51"><a href="#cb1-51" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-52"><a href="#cb1-52" aria-hidden="true" tabindex="-1"></a><span class="fu">### 03-07 | 18S and TCS Abstract</span></span>
<span id="cb1-53"><a href="#cb1-53" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-54"><a href="#cb1-54" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>A quick preview to the 18S phylogeny: <span class="co">[</span><span class="ot">genetic result</span><span class="co">](/notebooks/42%20Result%20-%20Genetics.qmd)</span></span>
<span id="cb1-55"><a href="#cb1-55" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Abstract for TCS Meeting Paris: <span class="co">[</span><span class="ot">TCS Abstract</span><span class="co">](/notebooks/11%20TCS-Abstract.qmd)</span></span>
<span id="cb1-56"><a href="#cb1-56" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>All drawing scans are <span class="co">[</span><span class="ot">here</span><span class="co">](/notebooks/51%20Drawing.qmd)</span></span>
<span id="cb1-57"><a href="#cb1-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-58"><a href="#cb1-58" aria-hidden="true" tabindex="-1"></a><span class="fu">### 02-28 | Loan request ZMH (Pending)</span></span>
<span id="cb1-59"><a href="#cb1-59" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-60"><a href="#cb1-60" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>**Type materials**</span>
<span id="cb1-61"><a href="#cb1-61" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-62"><a href="#cb1-62" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>*A. puertoricana*: ZMH K–46195 (Holotype), ZMH K–46196, ZMH K–46198 (Puerto Rico Trench)</span>
<span id="cb1-63"><a href="#cb1-63" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>*A. eleganta*: ZMH K–40716 (Holotype), ZMH K–40717, ZMH K–40718. (Weddell Sea)</span>
<span id="cb1-64"><a href="#cb1-64" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-65"><a href="#cb1-65" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>**ZMH K–46200**: *A. eleganta*</span>
<span id="cb1-66"><a href="#cb1-66" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-67"><a href="#cb1-67" aria-hidden="true" tabindex="-1"></a>*A. eleganta* morphologically identified by @malyutina2018, found at the same station as *A. puertoricana* (Puerto Rico Trench). A big support that *A. eleganta* and *A. puertoricana* are the same species if we can pull genetics and proteomics from them.</span>
<span id="cb1-68"><a href="#cb1-68" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-69"><a href="#cb1-69" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>**ZMH K–46199**: *A. annulatus*</span>
<span id="cb1-70"><a href="#cb1-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-71"><a href="#cb1-71" aria-hidden="true" tabindex="-1"></a>::: {.callout-note}</span>
<span id="cb1-72"><a href="#cb1-72" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-73"><a href="#cb1-73" aria-hidden="true" tabindex="-1"></a>@malyutina2018:</span>
<span id="cb1-74"><a href="#cb1-74" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-75"><a href="#cb1-75" aria-hidden="true" tabindex="-1"></a>Acanthocope annulatus Menzies, 1962 (Fig. 16)  </span>
<span id="cb1-76"><a href="#cb1-76" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-77"><a href="#cb1-77" aria-hidden="true" tabindex="-1"></a>Acanthocope annulatus (Menzies, 1962: 155, Fig. 44 B, C; Malyutina and Brandt, 2004: 11, Figs. 6–10.  Material examined: 1 male (5.9 mm TL), ZMH K–46199, Puerto Rico Trench abyssal area, RV Sonne, Vema-TRANSIT cruise, 25 January 2015, C-EBS, station 14-1, 19°1.63′N 67°9.73′W, 4552‒4552 m depth, supranet, bottom temperature: 2.25 °C, bottom O2: 261.13 μM, bottom current: 5.25 cm/s. </span>
<span id="cb1-78"><a href="#cb1-78" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-79"><a href="#cb1-79" aria-hidden="true" tabindex="-1"></a>Remarks: The only specimen of the species, found at the same station as A. puertoricana sp. nov., was identified as it has all described characters of the Acanthocope annulatus Menzies, 1962. This new record is located at more than 10000 km far away from the type locality in the southeastern Atlantic (SW of Cape Town). Future sampling in the Atlantic and genetic investigations might shed some light on the true distribution of this species.</span>
<span id="cb1-80"><a href="#cb1-80" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-81"><a href="#cb1-81" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb1-82"><a href="#cb1-82" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-83"><a href="#cb1-83" aria-hidden="true" tabindex="-1"></a><span class="fu">## Abstract</span></span>
<span id="cb1-84"><a href="#cb1-84" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-85"><a href="#cb1-85" aria-hidden="true" tabindex="-1"></a>{{&lt; include "notebooks/10 Abstract.qmd" &gt;}}</span></code><button title="Copy to Clipboard" class="code-copy-button" data-in-quarto-modal=""><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->




</body></html>