{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# \n",
    "\n",
    "## Phylogeny\n",
    "\n",
    "Sequence cleaning, reference mapping, alignment was done in Geneious. S\n",
    "\n",
    "### 18S\n",
    "\n",
    "1.  The 5’ and 3’ end of the MSA was first trimmed to a region that at least the half of the alignment has reached the position - to avoid included region where only one or two sequences are available.\n",
    "2.  MSA was trimmed by [Gblocks](https://home.cc.umanitoba.ca/~psgendb/doc/Castresana/Gblocks_documentation.html#Options) with following parameter:\n",
    "    -   Minimum Number Of Sequences For A Conserved Position: 9 (minimum)\n",
    "    -   Minimum Number Of Sequences For A Flank Position: 9\n",
    "    -   Maximum Number Of Contiguous Nonconserved Positions: 8 (default)\n",
    "    -   Minimum Length Of A Block: 2 (default: 10, manually to include more sites)\n",
    "    -   Allowed Gap Positions: ALL\n",
    "3.  [Gblocks result](assets/Phylogeny/Alignment.fasta-gb.html)\n",
    "4.  Trimmed MSA was tested by IQTREE for best fitting model \\> HKY model (corresponds to `lset nst = 2` in MrBayes)\n",
    "5.  A joint analysis with concatenated sequences (COI + Gblocks trimmed 18S) was done with following MrBayes command, which also corresponds to MrBayes command used for COI and 18S analysis.\n",
    "\n",
    "<!-- -->\n",
    "\n",
    "    BEGIN MrBayes;\n",
    "\n",
    "    set autoclose=yes nowarn=yes;\n",
    "\n",
    "    charset COI_pos1 = 1-623.\\3;\n",
    "    charset COI_pos2 = 2-623-.\\3;\n",
    "    charset COI_pos3 = 3-623.\\3;\n",
    "    charset 18S = 624-2663;\n",
    "    partition conc = 4: COI_pos1, COI_pos2, COI_pos3, 18S;\n",
    "\n",
    "    set partition=conc;\n",
    "\n",
    "    lset applyto=(1,2,3) nst=2 rates=invgamma code=Invermt ploidy=Haploid;\n",
    "    lset applyto=(4) nst=6 rates=invgamma;\n",
    "                                \n",
    "    mcmc ngen=5000000 nruns=6 nchains=6 relburnin=yes samplefreq=100 printfreq=10000 diagnfreq=10000 checkfreq=10000 savebrlens=yes starttree=random;\n",
    "                \n",
    "    sump burnin=500000;\n",
    "    sumt burnin=500000 contype=halfcompat;\n",
    "    report possel=yes siteomega=yes siterate=yes;\n",
    "\n",
    "    END;\n",
    "\n",
    "1.  *Vanhoeffenura sp.* is the only data from NCBI gene bank that has both 18S and COI from the same specimen (MB E1). COI: *EF682284.1*, 18S: *EF682239.1*."
   ],
   "id": "27b130f8-c248-4db0-9081-1cfa9a8be53f"
  },
  {
   "cell_type": "raw",
   "metadata": {
    "raw_mimetype": "text/html"
   },
   "source": [
    "<!-- -->"
   ],
   "id": "e4bed2f7-9fb0-4547-9d1d-b9610dde36cf"
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "```` markdown\n",
    "## Phylogeny\n",
    "\n",
    "Sequence cleaning, reference mapping, alignment was done in Geneious. S\n",
    "\n",
    "### 18S\n",
    "\n",
    "1. The 5' and 3' end of the MSA was first trimmed to a region that at least the half of the alignment has reached the position - to avoid included region where only one or two sequences are available.\n",
    "2. MSA was trimmed by [Gblocks](https://home.cc.umanitoba.ca/~psgendb/doc/Castresana/Gblocks_documentation.html#Options) with following parameter:\n",
    "    - Minimum Number Of Sequences For A Conserved Position: 9 (minimum)\n",
    "    - Minimum Number Of Sequences For A Flank Position: 9\n",
    "    - Maximum Number Of Contiguous Nonconserved Positions: 8 (default)\n",
    "    - Minimum Length Of A Block: 2 (default: 10, manually to include more sites)\n",
    "    - Allowed Gap Positions: ALL\n",
    "3. [Gblocks result](assets/Phylogeny/Alignment.fasta-gb.html)\n",
    "4. Trimmed MSA was tested by IQTREE for best fitting model > HKY model (corresponds to `lset nst = 2` in MrBayes)\n",
    "5. A joint analysis with concatenated sequences (COI + Gblocks trimmed 18S) was done with following MrBayes command, which also corresponds to MrBayes command used for COI and 18S analysis.\n",
    "\n",
    "```\n",
    "BEGIN MrBayes;\n",
    "\n",
    "set autoclose=yes nowarn=yes;\n",
    "\n",
    "charset COI_pos1 = 1-623.\\3;\n",
    "charset COI_pos2 = 2-623-.\\3;\n",
    "charset COI_pos3 = 3-623.\\3;\n",
    "charset 18S = 624-2663;\n",
    "partition conc = 4: COI_pos1, COI_pos2, COI_pos3, 18S;\n",
    "\n",
    "set partition=conc;\n",
    "\n",
    "lset applyto=(1,2,3) nst=2 rates=invgamma code=Invermt ploidy=Haploid;\n",
    "lset applyto=(4) nst=6 rates=invgamma;\n",
    "                            \n",
    "mcmc ngen=5000000 nruns=6 nchains=6 relburnin=yes samplefreq=100 printfreq=10000 diagnfreq=10000 checkfreq=10000 savebrlens=yes starttree=random;\n",
    "            \n",
    "sump burnin=500000;\n",
    "sumt burnin=500000 contype=halfcompat;\n",
    "report possel=yes siteomega=yes siterate=yes;\n",
    "\n",
    "END;\n",
    "```\n",
    "\n",
    "6. *Vanhoeffenura sp.* is the only data from NCBI gene bank that has both 18S and COI from the same specimen (MB E1). COI: *EF682284.1*, 18S: *EF682239.1*.\n",
    "````"
   ],
   "id": "3af7e396-7479-437a-8750-c2bcd810331f"
  }
 ],
 "nbformat": 4,
 "nbformat_minor": 5,
 "metadata": {}
}
